<body>
<sec id="Sec1" sec-type="introduction">
<title>Introduction</title>
<p id="Par3">The development of germ cells and their differentiation into gametes is crucial for the faithful transmission of both genetic and epigenetic information to the next generation. Primordial germ cells (PGCs) are the first germ cells to emerge in the post-implantation embryo. In mice, as few as 3−5 founder PGCs are specified at E6.0-6.5 and these undergo extensive proliferation, increasing to around 20,000 by E12.5<sup><xref ref-type="bibr" rid="CR1">1</xref>–<xref ref-type="bibr" rid="CR3">3</xref></sup>. To develop into functional gametes, PGCs must undergo a unique developmental program, repressing somatic genes and activating pluripotency and germ-cell-specific factors<sup><xref ref-type="bibr" rid="CR4">4</xref>,<xref ref-type="bibr" rid="CR5">5</xref></sup>. This entails extensive epigenetic reprogramming resulting in altered histone modifications and DNA demethylation which facilitates the erasure of genomic imprints, and reactivation of the inactive X-chromosome<sup><xref ref-type="bibr" rid="CR6">6</xref>,<xref ref-type="bibr" rid="CR7">7</xref></sup>. DNA demethylation has been proposed to occur by multiple mechanisms<sup><xref ref-type="bibr" rid="CR8">8</xref></sup>. A prevalent model describes a two-step process involving a passive demethylation phase in which DNA methylation is diluted by DNA replication (‘reprogramming step 1’) followed by active, enzymatic DNA demethylation that occurs upon colonization of the embryonic gonads (‘reprogramming step 2’)<sup><xref ref-type="bibr" rid="CR9">9</xref>–<xref ref-type="bibr" rid="CR14">14</xref></sup>. Thus, PGC development is highly dependent on DNA replication, both for lineage expansion and for epigenetic reprogramming.</p>
<p id="Par4">The replication of DNA can be hindered by various obstacles, such as chemical damage to the DNA molecule or DNA secondary structures<sup><xref ref-type="bibr" rid="CR15">15</xref></sup>. The failure to resolve these impediments can have catastrophic consequences for a cell. Incomplete or under-replication of the genome can block cell cycle progression, either directly or by activation of the DNA damage response (DDR) including cell cycle checkpoints<sup><xref ref-type="bibr" rid="CR16">16</xref></sup>. If this persists the cell may ultimately die. To combat these challenges, eukaryotes have evolved both DNA repair as well DNA damage tolerance (DDT) mechanisms. One route of DDT is error-prone translesion synthesis (TLS). TLS allows DNA replication to continue past impediments and facilitates the filling of gaps which remain at the end of S-phase<sup><xref ref-type="bibr" rid="CR17">17</xref></sup>. TLS utilizes specialized polymerases with active sites that can accommodate damaged or distorted DNA templates and which lack proofreading activity, hence increasing the risk of DNA mutagenesis<sup><xref ref-type="bibr" rid="CR18">18</xref>,<xref ref-type="bibr" rid="CR19">19</xref></sup>. In the germ cell compartment, ensuring that replication can proceed is of paramount importance to allow sufficient cellular expansion to guard against sterility. However, any increase in the mutagenicity of replication increases the risk of deleterious phenotypes and inherited disease in future generations.</p>
<p id="Par5">Recent genome wide association studies (GWAS) have implicated multiple DDR pathways as determinants of infertility in humans, however little is known about the underlying mechanism or if this requirement extends to the TLS pathway<sup><xref ref-type="bibr" rid="CR20">20</xref>–<xref ref-type="bibr" rid="CR22">22</xref></sup>. In this study, we find that TLS plays a crucial role in the development of embryonic germ cells in both humans and mice. Our results show that factors involved in sequential stages of TLS are essential for PGC development. In the absence of TLS, PGCs are specified in normal numbers but fail to expand, resulting in a &gt; 150-fold reduction. In contrast to the severe effect on the germline, somatic tissues of these mutants are unperturbed in their development with no discernible impact on homeostasis or survival. Consistent with a defect in TLS, mutant PGCs show reduced proliferation, accumulation at the G2/M phase of the cell cycle and increased markers of unresolved DNA damage. In addition, the loss of TLS prevents progression of the germ cell transcriptional program and results in failure of genome-wide DNA demethylation, an essential and conserved step in the development of mammalian embryonic germ cells. Our findings define a critical role of TLS specifically in germ cell development, safeguarding fertility and enabling successful PGC epigenetic reprogramming.</p>
</sec>
<sec id="Sec2" sec-type="results">
<title>Results</title>
<sec id="Sec3">
<title>REV1 is required for PGC development in human and mouse</title>
<p id="Par6">GWAS studies focused on infertility have revealed genes important for human germline development<sup><xref ref-type="bibr" rid="CR20">20</xref>–<xref ref-type="bibr" rid="CR22">22</xref></sup>. Notably, factors that deal with DNA damage are frequent hits. In agreement with these findings, the DDR has been implicated in germ cell production and maintenance. Loss of several DDR factors results in infertility due to failure during meiosis, likely as a result of the inability to resolve meiotic DNA double strand breaks (DSBs) and recombination intermediates<sup><xref ref-type="bibr" rid="CR23">23</xref>–<xref ref-type="bibr" rid="CR29">29</xref></sup>. However, it has been found that DDR pathways and factors including Fanconi Anemia crosslink repair, base excision repair and homologous recombination amongst others are also required for the embryonic development of germ cells<sup><xref ref-type="bibr" rid="CR21">21</xref>,<xref ref-type="bibr" rid="CR30">30</xref>–<xref ref-type="bibr" rid="CR38">38</xref></sup>. One of these factors, REV7, is involved in multiple DNA repair transactions including the repair of DNA DSBs, mitotic progression, and TLS<sup><xref ref-type="bibr" rid="CR39">39</xref>–<xref ref-type="bibr" rid="CR46">46</xref></sup>. Due to the requirement of various replication-coupled repair pathways in the embryonic germline, we set out to test if the TLS pathway is required for fertility. TLS is important for the completion of DNA replication, and we therefore hypothesized that it may play a role in ensuring successful genome duplication during highly proliferative stages of gametogenesis. As early embryonic germ cell development is particularly proliferative, we asked if TLS is required during primordial germ cell (PGC) development. We employed an in vitro model in which iPSCs are differentiated into human PGC-like cells (hPGCLCs) (Fig. <xref ref-type="fig" rid="Fig1">1a</xref>)<sup><xref ref-type="bibr" rid="CR47">47</xref></sup>. To study the potential roles of TLS in human germ cell development we focused on REV1, a core component of the TLS pathway<sup><xref ref-type="bibr" rid="CR48">48</xref>–<xref ref-type="bibr" rid="CR52">52</xref></sup>. REV1 was disrupted in human BTAG iPSCs carrying both the BLIMP1-tdTomato and TFAP2C-EGFP PGCLC reporters (Supplementary Fig. <xref ref-type="media" rid="MOESM1">1a</xref>)<sup><xref ref-type="bibr" rid="CR47">47</xref></sup>. Clones with successful disruption of REV1 were identified by PCR then validated by testing for hypersensitivity to mitomycin C (MMC) (Supplementary Fig. <xref ref-type="media" rid="MOESM1">1b, c</xref>). We first assessed if the loss of REV1 affected iPSC function. As REV1 plays a role in DNA replication we assessed proliferation of mutant iPSCs and found no difference between <italic>REV1</italic><sup><italic>-/-</italic></sup> and the parental line (Supplementary Fig. <xref ref-type="media" rid="MOESM1">1d</xref>). We also assessed the ability of <italic>REV1</italic><sup><italic>-/-</italic></sup> iPSCs to differentiate into different lineages by measuring embryoid body formation<sup><xref ref-type="bibr" rid="CR53">53</xref></sup>. We found that REV1 lines were able to differentiate into all three germ layers similarly to the parental wildtype line (Supplementary Fig. <xref ref-type="media" rid="MOESM1">1e</xref>). In contrast, we found that three independent <italic>REV1</italic><sup><italic>-/-</italic></sup> lines had a significant reduction in the ability to induce hPGCLCs, consistent with a role in embryonic germ cell development (Fig. <xref ref-type="fig" rid="Fig1">1b, c</xref> and Supplementary Fig. <xref ref-type="media" rid="MOESM1">1f</xref>). While the frequency of hPGCLCs generated was significantly reduced in the absence of REV1, the few induced hPGCLCs expressed the canonical germ cell markers SOX17, TFAP2C and OCT4 (Fig. <xref ref-type="fig" rid="Fig1">1d</xref>). The magnified inset shows the nuclear localization of each factor (Fig. <xref ref-type="fig" rid="Fig1">1d</xref>).<fig id="Fig1"><label>Fig. 1</label><caption><title>REV1-deficiency leads to defects in hPGCLC induction and infertility in mice.</title><p><bold>a</bold> Schematic of the hPGCLC differentiation protocol adapted from<sup><xref ref-type="bibr" rid="CR47">47</xref></sup>. <bold>b</bold> Representative flow cytometry plots and (<bold>c</bold>) quantification of hPGCLC frequency at day 4 from three independent wildtype or <italic>REV1</italic><sup><italic>-/-</italic></sup> clones differentiated three times. (Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test). <bold>d</bold> Representative images of wildtype and <italic>REV1</italic><sup><italic>-/-</italic></sup> aggregates immunostained for SOX17, TFAP2C and OCT4. <bold>e</bold> Offspring when male (left) and female (right) wildtype or <italic>Rev1</italic><sup><italic>-/-</italic></sup> mice were mated with wildtype mates of the opposite sex (<italic>n</italic> = 6 mice per genotype, 3 per sex. Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test). <bold>f</bold> Quantification of testis mass of 8−12-week-old wildtype and <italic>Rev1</italic><sup><italic>-/-</italic></sup> mice (<italic>n</italic> = 12, 2, left to right. Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test). <bold>g</bold> Quantification of ovary mass from 8−12-week-old wildtype and <italic>Rev1</italic><sup><italic>-/-</italic></sup> mice (<italic>n</italic> = 13, 2, left to right. Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test). <bold>h</bold> H&amp;E-stained ovaries and testes seminiferous tubules from 8−12-week-old wildtype and <italic>Rev1</italic><sup><italic>-/-</italic></sup> mice. <bold>i</bold> Quantification of SCO tubules per section of testis of 8−12-week-old wildtype and <italic>Rev1</italic><sup><italic>-/-</italic></sup> mice (<italic>n</italic> = 10 and 4 animals, left to right. Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test). <bold>j</bold> Quantification of follicles per section of ovary from 8−12-week-old wildtype and <italic>Rev1</italic><sup><italic>-/-</italic></sup> mice (<italic>n</italic> = 8 and 3 animals, left to right). Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test. <bold>k</bold> Schematic for generation of <italic>Rev1</italic><sup><italic>-/-</italic></sup> embryos harboring the GOF18-GFP reporter. <bold>l</bold> GFP fluorescence images of gonads from wildtype and <italic>Rev1</italic><sup><italic>-/-</italic></sup> E12.5 embryos. <bold>m</bold> Representative flow cytometry plots and <bold>(n)</bold> quantification of PGCs by flow cytometry from wildtype and <italic>Rev1</italic><sup><italic>-/-</italic></sup> E12.5 embryos (<italic>n</italic> = 35 and 12, left to right. Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test).</p></caption><graphic id="d33e704" xlink:href="41467_2024_47219_Fig1_HTML"></graphic></fig></p>
<p id="Par7">Due to limitations of studying human gametogenesis in vivo, we asked if the requirement for TLS factors in PGC development was conserved to mice, thus facilitating mechanistic studies. REV1-deficient mice were crossed with wildtype mates to assess fertility. Consistent with our observations in human, we found that neither male nor female <italic>Rev1</italic><sup><italic>-/-</italic></sup> mice gave rise to offspring despite evidence of copulation (Fig. <xref ref-type="fig" rid="Fig1">1e</xref>)<sup><xref ref-type="bibr" rid="CR52">52</xref></sup>. Upon analysis of the gonads, we observed a striking reduction in testis mass but the mass of the ovaries was unaffected (Fig. <xref ref-type="fig" rid="Fig1">1f, g</xref>). Histological analysis of the testes revealed a majority (96.8%) of Sertoli-cell-only (SCO) seminiferous tubules and an absence of promyelocytic leukemia zinc finger (PLZF)<sup>+</sup> cells, which marks undifferentiated spermatogonial stem cells (SSCs, Fig. <xref ref-type="fig" rid="Fig1">1h, i</xref> and Supplementary Fig. <xref ref-type="media" rid="MOESM1">1g</xref>)<sup><xref ref-type="bibr" rid="CR54">54</xref></sup>. In females, we observed no follicles upon histological examination of <italic>Rev1</italic><sup><italic>-/-</italic></sup> ovaries, suggesting a complete failure of oogenesis (Fig. <xref ref-type="fig" rid="Fig1">1h, j</xref>). Together, these data reveal a lack of gametes in both <italic>Rev1</italic><sup><italic>-/-</italic></sup> male and female mice. The lack of visible PLZF<sup>+</sup> cells in male gonads and meiotic cells in either sex argues that the defect is pre-meiotic which is consistent with the hPGCLC data suggesting a failure in germ cell development during embryonic stages. Therefore, we studied the development of embryonic germ cells using the GOF18-GFP PGC reporter in which GFP expression is driven by a fragment of the <italic>Oct4</italic> promoter (<italic>Oct4ΔPE</italic>) (Fig. <xref ref-type="fig" rid="Fig1">1k</xref>)<sup><xref ref-type="bibr" rid="CR55">55</xref>,<xref ref-type="bibr" rid="CR56">56</xref></sup>. We intercrossed <italic>Rev1</italic><sup><italic>+/-</italic></sup> mice carrying the GOF18-GFP PGC reporter and harvested the embryonic gonads at E12.5, prior to sexually divergent germline development. Imaging of the gonads revealed a dramatic reduction in GOF18-GFP<sup>+</sup> cells in <italic>Rev1</italic><sup><italic>-/-</italic></sup> embryos compared to wildtype controls, consistent with the hPGCLC data (Fig. <xref ref-type="fig" rid="Fig1">1l</xref>). Quantification of PGCs (double positive for the marker SSEA-1 (stage specific embryonic antigen-1) and GFP; SSEA1<sup>+</sup>GOF18-GFP<sup>+</sup>) by flow cytometry revealed a significant reduction in <italic>Rev1</italic><sup><italic>-/-</italic></sup> embryos (Fig. <xref ref-type="fig" rid="Fig1">1m, n</xref>). Together, these data show that the TLS factor REV1 plays a key role in embryonic germ cell development in both humans and mice.</p>
</sec>
<sec id="Sec4">
<title>The requirement for the C-terminal domain of REV1 and the Polζ subunit REV7 links TLS to PGC development</title>
<p id="Par8">REV1 can act enzymatically through its deoxycytidyl transferase activity which can drive mutagenesis and is important for immunoglobulin diversification<sup><xref ref-type="bibr" rid="CR48">48</xref>,<xref ref-type="bibr" rid="CR50">50</xref>,<xref ref-type="bibr" rid="CR51">51</xref></sup>. Alternatively, it can act as a protein-scaffold through its C-terminal protein interaction domain which is able to recruit TLS factors to sites of lesion bypass<sup><xref ref-type="bibr" rid="CR57">57</xref></sup>. This region of REV1 plays a crucial role in maintaining cellular resistance to DNA damage that impedes replication indicating that the C-terminus is the key region of REV1 for TLS transactions. To determine if the catalytic or recruitment function of REV1 is required for PGC development, we generated a catalytically dead allele of <italic>Rev1</italic>, in which the catalytic residues D568 and E569 were mutated to alanine (<italic>Rev1</italic><sup><italic>AA</italic></sup>) (Fig. <xref ref-type="fig" rid="Fig2">2a</xref> and Supplementary Fig. <xref ref-type="media" rid="MOESM1">2a</xref>)<sup><xref ref-type="bibr" rid="CR58">58</xref></sup>. We also generated mice with a mutant <italic>Rev1</italic> allele in which the DNA encoding the final 100 amino acids of REV1 was deleted (<italic>Rev1</italic><sup><italic>CT</italic></sup>) (Fig. <xref ref-type="fig" rid="Fig2">2a</xref> and Supplementary Fig. <xref ref-type="media" rid="MOESM1">2b, c</xref>)<sup><xref ref-type="bibr" rid="CR57">57</xref></sup>. To confirm the stability of the protein product of the <italic>Rev1</italic><sup><italic>CT</italic></sup> allele, we expressed N-terminally FLAG-tagged full length or C-terminally truncated REV1 in cell lines and performed immunoblotting. This revealed comparable protein levels and a lack of degradation products (Supplementary Fig. <xref ref-type="media" rid="MOESM1">2d</xref>). Furthermore, immunofluorescence analysis of cell lines expressing GFP-tagged full length and C-terminally truncated REV1 showed localization of both to the nucleus (Supplementary Fig. <xref ref-type="media" rid="MOESM1">2e</xref>). To further validate these <italic>Rev1</italic> mutant alleles, we derived cell lines from mice and measured mRNA expression and cellular sensitivity to DNA damaging agents. The mRNA expression of mutant alleles was comparable to that of <italic>Rev1</italic> in wildtype cells (Supplementary Fig. <xref ref-type="media" rid="MOESM1">2f</xref>). Moreover, the C-terminus but not catalytic activity of REV1 was required to overcome replication blocking DNA damage, in line with previous reports (Supplementary Fig. <xref ref-type="media" rid="MOESM1">2g</xref>)<sup><xref ref-type="bibr" rid="CR57">57</xref>,<xref ref-type="bibr" rid="CR59">59</xref></sup>.<fig id="Fig2"><label>Fig. 2</label><caption><title>REV7 is required for PGC development in mice.</title><p><bold>a</bold> Schematic of wildtype (<italic>Rev1</italic><sup><italic>+</italic></sup>), null (<italic>Rev1</italic><sup>-</sup>), catalytically inactive REV1 (<italic>Rev1</italic><sup><italic>AA</italic></sup>) and C-terminally truncated REV1 (<italic>Rev1</italic><sup><italic>CT</italic></sup>) alleles. <bold>b</bold> GFP fluorescence images of gonads from wildtype, <italic>Rev1</italic><sup><italic>-/-</italic></sup>, <italic>Rev1</italic><sup><italic>-/AA</italic></sup> and <italic>Rev1</italic><sup><italic>-/CT</italic></sup> E12.5 embryos. <bold>c</bold> Quantification of PGCs by flow cytometry from wildtype, <italic>Rev1</italic><sup><italic>-/-</italic></sup>, <italic>Rev1</italic><sup><italic>-/AA</italic></sup> and <italic>Rev1</italic><sup><italic>-/CT</italic></sup> embryos at E12.5 (<italic>n</italic> = 35, 12, 10 and 7, left to right. Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test). <bold>d</bold> H&amp;E-stained ovaries and PLZF-stained testes and quantification (<bold>e</bold>) of follicles per section of ovary (Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test). or (<bold>f</bold>) frequency of PLZF<sup>+</sup> cells per seminiferous tubule of 8−12-week-old wildtype, <italic>Rev1</italic><sup><italic>-/-</italic></sup> and <italic>Rev1</italic><sup><italic>-/AA</italic></sup> mice (the data shown represent the median and interquartile range; <italic>n</italic> = 150 tubules per genotype, 50 per genotype, <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test). <bold>g</bold> Droplet digital PCR (ddPCR) gene expression analysis of <italic>Rev1</italic>, <italic>Rev7, Polk</italic> and <italic>Polq</italic> in FACS-purified PGCs and surrounding somatic cells (SSEA1<sup>-</sup>GOF18-GFP<sup>−</sup>) from E10.5 embryos (<italic>n</italic> = 3 independent embryos. Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test). <bold>h</bold> GFP fluorescence images of gonads from wildtype, <italic>Rev7</italic><sup><italic>-/-</italic></sup>, <italic>Polk</italic><sup><italic>-/-</italic></sup> and <italic>Polq</italic><sup><italic>-/-</italic></sup> E12.5 embryos. <bold>i</bold> Quantification of PGCs from wildtype, <italic>Rev7</italic><sup><italic>-/-</italic></sup>, <italic>Polk</italic><sup><italic>-/-</italic></sup> and <italic>Polq</italic><sup><italic>-/-</italic></sup> E12.5 embryos by flow cytometry (<italic>n</italic> = 35, 14, 8 and 9, left to right. Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test). <bold>j</bold> H&amp;E-stained testis seminiferous tubules from 8−12-week-old wildtype and mutant mice. <bold>k</bold> Quantification of SCO tubules per section of testis of 8−12-week-old mice (<italic>n</italic> = 10, 7, 8 and 14, left to right. Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test). <bold>l</bold> H&amp;E-stained ovaries and (<bold>m</bold>) quantification of follicles per section of ovary from 8−12-week-old mice (<italic>n</italic> = 8, 3, 4 and 11, left to right. Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test).</p></caption><graphic id="d33e1130" xlink:href="41467_2024_47219_Fig2_HTML"></graphic></fig></p>
<p id="Par9">We next crossed the <italic>Rev1</italic><sup><italic>AA</italic></sup> or <italic>Rev1</italic><sup><italic>CT</italic></sup> alleles with <italic>Rev1</italic><sup><italic>+/-</italic></sup> mice carrying the GOF18-GFP PGC reporter. At E12.5, we found that <italic>Rev1</italic><sup><italic>-/AA</italic></sup> embryos had a reduction in the frequency of PGCs (median wildtype = 1383, median <italic>Rev1</italic><sup><italic>-/AA</italic></sup> = 884, Fig. <xref ref-type="fig" rid="Fig2">2b, c</xref>). However, this was moderate when compared to <italic>Rev1</italic><sup><italic>-/CT</italic></sup> or <italic>Rev1</italic><sup><italic>-/-</italic></sup> embryos. Furthermore, <italic>Rev1</italic><sup><italic>-/AA</italic></sup> PGCs generated mature gametes in the gonads of adult mice which were competent in giving rise to viable offspring (Fig. <xref ref-type="fig" rid="Fig2">2d−f</xref> and Supplementary Fig. <xref ref-type="media" rid="MOESM1">2h</xref>). In contrast, E12.5 <italic>Rev1</italic><sup><italic>-/CT</italic></sup> embryos showed a significant reduction in the number of PGCs, with similar numbers to those in <italic>Rev1</italic><sup><italic>-/-</italic></sup> embryos (median = 27 and 20, respectively, Fig. <xref ref-type="fig" rid="Fig2">2b, c</xref>). Together these data show that the catalytic activity of REV1 plays a moderate role in germ cell development whilst the C-terminus, which coordinates protein-protein interactions during TLS, is critical for PGC development.</p>
<p id="Par10">To investigate the function of TLS in PGC development further, we investigated known REV1 interactors. First, we measured the expression of a subset of interactors (REV7, POLΚ and POLΘ) and found that like <italic>Rev1</italic>, each was more highly expressed in PGCs than in the surrounding somatic cells (Fig. <xref ref-type="fig" rid="Fig2">2g</xref>) (<italic>29, 37</italic>). To determine if these factors are also required for PGC development, we utilized previously generated and validated genetic models to generate E12.5 embryos deficient in each factor expressing the GOF18-GFP PGC reporter<sup><xref ref-type="bibr" rid="CR45">45</xref>,<xref ref-type="bibr" rid="CR60">60</xref>,<xref ref-type="bibr" rid="CR61">61</xref></sup>. Whilst <italic>Polk</italic><sup><italic>-/-</italic></sup> and <italic>Polq</italic><sup><italic>-/-</italic></sup> embryos had comparable numbers of PGCs to wildtype, REV7-deficient embryos had a significant reduction similar to that observed in <italic>Rev1</italic><sup><italic>-/-</italic></sup> embryos (Fig. <xref ref-type="fig" rid="Fig2">2h, i</xref>). Consistent with this, the profound reduction in the number of PGCs in <italic>Rev7</italic><sup><italic>-/-</italic></sup> embryos resulted in adult gonads devoid of germ cells (Fig. <xref ref-type="fig" rid="Fig2">2j−m</xref> and Supplementary Fig. <xref ref-type="media" rid="MOESM1">3a</xref>).</p>
<p id="Par11">Together these data reveal that it is the C-terminal protein interaction domain of REV1 and its interactor REV7 that are required during PGC development. REV7/repro22 has previously been identified in an ENU screen as a factor needed for fertility and subsequent studies have shown REV7 to be important during PGC expansion<sup><xref ref-type="bibr" rid="CR39">39</xref>–<xref ref-type="bibr" rid="CR41">41</xref></sup>. REV7 is the non-catalytic subunit of Polζ, a B-family polymerase that extends the nascent strand following lesion bypass before handing back to replicative polymerases<sup><xref ref-type="bibr" rid="CR62">62</xref>,<xref ref-type="bibr" rid="CR63">63</xref></sup>. As only Polζ has such an activity it is considered essential for TLS transactions. However, REV7 also has non-TLS roles in DSB repair, cell cycle regulation and the shelterin complex<sup><xref ref-type="bibr" rid="CR42">42</xref>–<xref ref-type="bibr" rid="CR46">46</xref></sup>. For the polymerase activity of Polζ, REV7 must bind to REV3L which is not required for the other functions of REV7<sup><xref ref-type="bibr" rid="CR64">64</xref></sup>. As REV3L is essential for embryonic development any potential role in PGC development cannot be tested<sup><xref ref-type="bibr" rid="CR65">65</xref>–<xref ref-type="bibr" rid="CR67">67</xref></sup>. However, we find that <italic>Rev3l</italic> and <italic>Rev7</italic> have similar patterns of expression in PGCs at E10.5 which mimics that of <italic>Rev1</italic> (Supplementary Fig. <xref ref-type="media" rid="MOESM1">3b</xref>). Crucially, Polζ is recruited to sites of lesion bypass for DNA synthesis through physical interactions between the REV7 subunit and REV1’s C-terminal domain<sup><xref ref-type="bibr" rid="CR46">46</xref></sup>. Hence, our discovery that both the C-terminus of REV1 and the REV7 subunit of Polζ are required in PGCs argues that the common function of both - in TLS - is needed during germ cell development.</p>
</sec>
<sec id="Sec5">
<title>The post-translational modification of PCNA preserves PGC development</title>
<p id="Par12">If TLS is critical for the development of PGCs, we hypothesized that PCNA ought to play an important role. PCNA is an essential component of the replisome that links DNA repair and DDT responses to replication<sup><xref ref-type="bibr" rid="CR68">68</xref></sup>. Upon stalling of replication forks, lysine 164 (K164) of PCNA can be sumoylated, mono-, or poly-ubiquitinated to engage DDT pathways<sup><xref ref-type="bibr" rid="CR69">69</xref>–<xref ref-type="bibr" rid="CR71">71</xref></sup>. Monoubiquitination of PCNA at K164 is required for TLS, facilitating the recruitment of specialized polymerases to sites of replication blocking lesions, enabling their bypass<sup><xref ref-type="bibr" rid="CR72">72</xref>–<xref ref-type="bibr" rid="CR74">74</xref></sup>. As our results found a requirement for the TLS factors REV1 and REV7, we wanted to study if there is also a role for PCNA K164 modification during PGC development. We used CRISPR/Cas9 to generate mice in which K164 of PCNA is mutated to arginine (<italic>Pcna</italic><sup><italic>K164R</italic></sup>) (Supplementary Fig. <xref ref-type="media" rid="MOESM1">4a</xref>). In this model, PCNA K164-modification dependent DDT, including TLS, is abolished<sup><xref ref-type="bibr" rid="CR68">68</xref></sup>. We first set out to determine if our allele recapitulated features of previously described mice carrying the PCNA K164R mutation (<italic>Pcna</italic><sup><italic>tm1Jcbs</italic></sup> MGI:3761720 and <italic>Tg</italic><sup><italic>(Pcna*K164)1Mdsc</italic></sup> MGI:97503)<sup><xref ref-type="bibr" rid="CR75">75</xref>,<xref ref-type="bibr" rid="CR76">76</xref></sup>. As expected, <italic>Pcna</italic><sup><italic>K164R/K164R</italic></sup> (herein referred to as <italic>Pcna</italic><sup><italic>R/R</italic></sup>) mouse embryonic fibroblasts (MEFs) derived from our allele were hypersensitive to ultraviolet irradiation (Supplementary Fig. <xref ref-type="media" rid="MOESM1">4b</xref>)<sup><xref ref-type="bibr" rid="CR75">75</xref></sup>. However, despite PCNA K164 being important for the cellular response to replication blocking DNA damage, <italic>Pcna</italic><sup><italic>R/R</italic></sup> mice were born at the expected Mendelian ratios with adult mice having similar lifespan to wildtype littermates (Supplementary Fig. <xref ref-type="media" rid="MOESM1">4c, d</xref>). Consistent with previously published data, we found that the gonads of homozygous adult mice were smaller than wildtype and that both male and female <italic>Pcna</italic><sup><italic>R/R</italic></sup> mice were sterile (Fig. <xref ref-type="fig" rid="Fig3">3a−d</xref>)<sup><xref ref-type="bibr" rid="CR75">75</xref></sup>. Histological analysis revealed that the majority (98.2%) of the testes contained SCO seminiferous tubules and that the ovaries were devoid of follicles (Fig. <xref ref-type="fig" rid="Fig3">3e−h</xref>). Similar to REV1-deficiency, the failure in gametogenesis began during embryonic development (Fig. <xref ref-type="fig" rid="Fig3">3i, j</xref>). At E12.5, the numbers of PGCs observed in <italic>Pcna</italic><sup><italic>R/R</italic></sup> embryos was similar to <italic>Rev1</italic><sup><italic>-/-</italic></sup> with a &gt; 150-fold reduction when compared to wildtype (Fig. <xref ref-type="fig" rid="Fig3">3k</xref>). To study if there were sex-specific differences in the embryonic germ cell defect of TLS-deficient embryos, we compared E12.5 male and female <italic>Pcna</italic><sup><italic>R/R</italic></sup> embryos. We found comparable PGC numbers in both sexes, confirming a common defect (Supplementary Fig. <xref ref-type="media" rid="MOESM1">4e</xref>).<fig id="Fig3"><label>Fig. 3</label><caption><title>Embryonic origin of sterility upon PCNA K164 mutation.</title><p><bold>a</bold> Representative images of testes and ovaries from wildtype and <italic>Pcna</italic><sup><italic>R/R</italic></sup> mice.Quantification of (<bold>b</bold>) testicular or (<bold>c</bold>) ovarian mass from 8-12-week-old wildtype and <italic>Pcna</italic><sup><italic>R/R</italic></sup> mice (<italic>n</italic> = 12, 10, 13 and 8 left to right. Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test). <bold>d</bold> Cumulative number of offspring when wildtype or <italic>Pcna</italic><sup><italic>R/R</italic></sup> mice were mated with wildtype mates of the opposite sex (<italic>n</italic> = 6 mice per genotype, 3 per sex. Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test). <bold>e</bold> H&amp;E-stained testis seminiferous tubules and (<bold>f</bold>) quantification of SCO tubules per section of testis of 8−12-week-old wildtype and <italic>Pcna</italic><sup><italic>R/R</italic></sup> (<italic>n</italic> = 10 and 15, left to right. Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test). <bold>g</bold> H&amp;E-stained ovaries and (<bold>h</bold>) quantification of follicles per section of ovary from 8−12-week-old wildtype and <italic>Pcna</italic><sup><italic>R/R</italic></sup> mice (<italic>n</italic> = 8 and 11, left to right. Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test). <bold>i</bold> GFP fluorescence images of gonads from wildtype and <italic>Pcna</italic><sup><italic>R/R</italic></sup> E12.5 embryos. <bold>j</bold> Representative flow cytometry plots and (<bold>k</bold>) quantification of PGCs from wildtype, <italic>Rev1</italic><sup><italic>-/-</italic></sup> and <italic>Pcna</italic><sup><italic>R/R</italic></sup> gonads at E12.5 (<italic>n</italic> = 35, 12 and 21, left to right. Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test). <bold>l</bold> Quantification of PGCs by flow cytometry from wildtype, <italic>Rev1</italic><sup><italic>-/-</italic></sup>, <italic>Pcna</italic><sup><italic>R/R</italic></sup> and <italic>Rev7</italic><sup><italic>-/-</italic></sup> embryos at E8.5 (<italic>n</italic> = 18, 5, 2 and 3, left to right. Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test). <bold>m</bold> Quantification of PGCs by flow cytometry from E8.5 to E12.5 in wildtype and mutant embryos (wildtype, <italic>n</italic> = 18, 12, 9, 3 and 17; <italic>Rev1</italic><sup><italic>−/−</italic></sup>, <italic>n</italic> = 5, 6, 5, 2 and 8; <italic>Pcna</italic><sup><italic>R/R</italic></sup>, <italic>n</italic> = 2, 4, 4, 7 and 9; <italic>Rev7</italic><sup><italic>−/−</italic></sup>, I = 3, 0, 0, 0 and 7, independent embryos, left to right. Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test).</p></caption><graphic id="d33e1653" xlink:href="41467_2024_47219_Fig3_HTML"></graphic></fig></p>
<p id="Par13">These findings led us to investigate the timing of the PGC defect in <italic>Rev1</italic><sup><italic>-/-</italic></sup>, <italic>Rev7</italic><sup><italic>-/-</italic></sup> and <italic>Pcna</italic><sup><italic>R/R</italic></sup> embryos more systematically. The PGCs in mouse embryos are specified between E6.0-6.5 then start to migrate and extensively proliferate from E8.5<sup><xref ref-type="bibr" rid="CR1">1</xref></sup>. We first quantified the number of PGCs at E8.5 and found no significant difference in the number of PGCs between wildtype and the three TLS mutants (Fig. <xref ref-type="fig" rid="Fig3">3l</xref>). We confirmed this result using an alternative PGC-reporter, <italic>Stella-GFP</italic> (Supplementary Fig. <xref ref-type="media" rid="MOESM1">4f</xref>)<sup><xref ref-type="bibr" rid="CR77">77</xref></sup>. Next, we quantified the number of PGCs by flow cytometry at each day of development between E9.5-12.5, replotting the E8.5 data on the same axis, and found that from E9.5 onwards <italic>Rev1</italic><sup><italic>-/-</italic></sup> and <italic>Pcna</italic><sup><italic>R/R</italic></sup> embryos had a significantly contracted PGC pool when compared to wildtype (Fig. <xref ref-type="fig" rid="Fig3">3m</xref> and Supplementary Fig. <xref ref-type="media" rid="MOESM1">4g−i</xref>). These data reveal that in addition to similar magnitudes, the timing of PGC defect is comparable across the TLS mutants.</p>
<p id="Par14">Having observed an essential requirement for TLS factors in mammalian germ cell development, we wondered if this was unique to the germline. In agreement with a previous report, we found a 2.6-fold reduction in the frequency of hematopoietic stem and progenitor cells (HSPCs) in adult <italic>Pcna</italic><sup><italic>R/R</italic></sup> mice (Supplementary Fig. <xref ref-type="media" rid="MOESM1">5a, b</xref>)<sup><xref ref-type="bibr" rid="CR78">78</xref></sup>. We went on to assess genome stability in the hematopoietic compartment by measuring the frequency of micronucleated normochromic erythrocytes (Mn-NCE). We found a significant increase in the frequency of Mn-NCE in <italic>Pcna</italic><sup><italic>R/R</italic></sup> mice when compared to wildtype (Supplementary Fig. <xref ref-type="media" rid="MOESM1">5c</xref>). However, there was no change in the frequency of peripheral red blood cells or hemoglobin concentration (Supplementary Fig. <xref ref-type="media" rid="MOESM1">5d, e</xref>). As adult HSCs are largely quiescent we hypothesized that the defect may begin during embryonic development when HSPCs are highly proliferative<sup><xref ref-type="bibr" rid="CR79">79</xref></sup>. Similar to adult mice, we found that E12.5 <italic>Pcna</italic><sup><italic>R/R</italic></sup> embryos had a 1.73-fold reduction in the frequency of HSPCs, revealing that the blood stem cell defect begins during embryonic development (Supplementary Fig. <xref ref-type="media" rid="MOESM1">5f</xref>). Despite a numerical HSPC defect and evidence of genomic instability, we found that blood compartment homeostasis was remarkably intact with adult <italic>Pcna</italic><sup><italic>R/R</italic></sup> mice showing comparable bone marrow cellularity to littermates, normal blood cell maturation, and sustained peripheral blood homeostasis (Supplementary Fig. <xref ref-type="media" rid="MOESM1">5g−l</xref>).</p>
<p id="Par15">We next examined a panel of vital somatic tissues and found no gross histological abnormalities consistent with the normal longevity of <italic>Pcna</italic><sup><italic>R/R</italic></sup> mice (Supplementary Fig. <xref ref-type="media" rid="MOESM1">6a</xref>). As PCNA is an essential component of the replication machinery, and its modification at K164 directly couples DDT to the replisome, we assessed the proportion of actively dividing cells in highly mitotic tissues. We found comparable numbers of Ki67<sup>+</sup> cells in the bone marrow, crypts of the ileum and hair follicle bulges of the skin of <italic>Pcna</italic><sup><italic>R/R</italic></sup> mice compared to controls (Supplementary Fig. <xref ref-type="media" rid="MOESM1">6b−d</xref>). Furthermore, we also assessed the effect of PCNA K164R mutation on the function of the liver and kidney and found no difference compared to wildtype (Supplementary Fig. <xref ref-type="media" rid="MOESM1">6e−h</xref>). These data show that the K164R mutation of PCNA does not lead to a global reduction in proliferation nor to loss of tissue homeostasis or function. In contrast, histological analysis of the adult gonads revealed complete loss of homeostasis (Supplementary Fig. <xref ref-type="media" rid="MOESM1">7</xref>). Consistent with the histological analysis, we observe loss of tissue function in mutants leading to disruption of the hypothalamic-pituitary-gonadal axis. This led to systemic hormonal dysregulation with elevated levels of luteinising hormone (LH) and follicle stimulating hormone (FSH) driving reactive stromal hyperplasia and a significant increase in testicular interstitial cells and ovarian stroma. As animals age, persistent hypergonadotrophism leads to a substantial increase in ovarian mass with mutant ovaries of 12-month-old mice being 3 times larger than wildtype controls (Supplementary Fig. <xref ref-type="media" rid="MOESM1">7d</xref>). These features are consistent across TLS-deficient mice with similar defects seen in <italic>Rev1</italic><sup><italic>-/-</italic></sup> and <italic>Rev7</italic><sup><italic>-/-</italic></sup> adults (Supplementary Fig. <xref ref-type="media" rid="MOESM1">7</xref>).</p>
<p id="Par16">Overall, whilst the PCNA K164R mutation results in increased sensitivity to exogenous DNA damage and a reduction in HSPCs, our findings reveal that the development and homeostasis of somatic tissues are largely unaffected. In contrast, the modification site of PCNA is essential for PGC development and fertility. Failure of embryonic germ cell development results in dysregulation of hypothalamic-pituitary-gonadal hormone regulation with inappropriate stromal proliferation in adults. The temporality and magnitude of <italic>Rev1</italic><sup><italic>-/-</italic></sup>, <italic>Rev7</italic><sup><italic>-/-</italic></sup> and <italic>Pcna</italic><sup><italic>R/R</italic></sup> germ cell defects and the molecular dissection of REV1 suggest a common origin of the defect - likely their shared role in TLS.</p>
</sec>
<sec id="Sec6">
<title>Loss of genome stability and reduced proliferation of Pcna<sup>R/R</sup> and Rev1<sup>-/-</sup> PGCs</title>
<p id="Par17">Next, we investigated the mechanisms of PGC failure in the absence of TLS factors. As TLS mitigates replication blocks, we asked if <italic>Pcna</italic><sup><italic>R/R</italic></sup> or <italic>Rev1</italic><sup><italic>-/-</italic></sup> PGCs accumulate unresolved DNA damage. We stained PGCs for the phosphorylation of histone variant H2A.X (ɣ-H2A.X), a marker of DNA DSBs<sup><xref ref-type="bibr" rid="CR80">80</xref></sup>. A greater proportion of TLS-deficient PGCs had &gt;10 ɣ-H2A.X foci compared to wildtype (Fig. <xref ref-type="fig" rid="Fig4">4a</xref>). Consistent with our previous data this finding was specific to PGCs as an increase was not observed in the surrounding somatic tissue (Supplementary Fig. <xref ref-type="media" rid="MOESM1">8a</xref>). To build on this, we stained for the presence of RPA foci, which binds to single stranded DNA generated during repair transactions or when DNA replication is perturbed and leads to under-replicated regions in the genome<sup><xref ref-type="bibr" rid="CR81">81</xref></sup>. We stained for the RPA subunit RPA32 and quantified the frequency of PGCs harboring RPA foci as this is indicative of DNA damage<sup><xref ref-type="bibr" rid="CR82">82</xref>,<xref ref-type="bibr" rid="CR83">83</xref></sup>. Consistent with the results from the ɣ-H2A.X staining we observed a higher frequency of PGCs with nuclear RPA foci in <italic>Pcna</italic><sup><italic>R/R</italic></sup> and <italic>Rev1</italic><sup><italic>-/-</italic></sup> embryos (Fig. <xref ref-type="fig" rid="Fig4">4b</xref>). We went on to assess if the DDR was activated by measuring threonine 68 (Thr68) phosphorylation of CHK2, the critical checkpoint kinase<sup><xref ref-type="bibr" rid="CR84">84</xref></sup>. We found that an increased frequency of <italic>Pcna</italic><sup><italic>R/R</italic></sup> and <italic>Rev1</italic><sup><italic>-/-</italic></sup> PGCs stained positive for pCHK2 when compared to wildtype (Fig. <xref ref-type="fig" rid="Fig4">4c</xref>). This shows that PGC development requires TLS and that in its absence cells accumulate damaged DNA.<fig id="Fig4"><label>Fig. 4</label><caption><title>Genome instability and cell cycle abnormalities in PcnaR/R and Rev1-/- PGCs.</title><p><bold>a</bold> Top: Representative images of ɣ-H2A.X foci in the nucleus of PGCs (GOF18-GFP<sup>+</sup>) at E12.5. Bottom: Frequency of PGCs with &gt;10 ɣ-H2A.X foci per nucleus (<italic>n</italic> = 6, 5 and 3, left to right. Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test). <bold>b</bold> Top: Representative images of RPA foci in the nucleus of PGCs at E12.5. Bottom: Frequency of PGCs with RPA foci (<italic>n</italic> = 5, 4 and 3, left to right. Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test). <bold>c</bold> Top: Representative images of E12.5 gonads stained for phosphorylation of CHK2 kinase at residue Thr68 (pCHK2). Bottom: Frequency of PGCs that stain positive for pCHK2 (<italic>n</italic> = 5, 3 and 3, left to right. Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test). <bold>d</bold> Top: Representative images of E12.5 gonads stained for cleaved-Caspase-3 (CC3) and GFP. Bottom: Frequency of PGCs that stain positive for CC3 (<italic>n</italic> = 8, 4 and 3, left to right. Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test). <bold>e</bold> Top: Representative images of E12.5 gonads stained for EdU and GFP. Bottom: Frequency of PGCs that stain positive for EdU (<italic>n</italic> = 3 per genotype. Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test). <bold>f</bold> Top: Representative images of E12.5 gonads stained for phosphorylated-histone-H3 (pH3) and GFP. Bottom: Frequency of PGCs that stain positive for pH3 (<italic>n</italic> = 11, 4 and 2, left to right. Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test).</p></caption><graphic id="d33e1994" xlink:href="41467_2024_47219_Fig4_HTML"></graphic></fig></p>
<p id="Par18">Altered proliferation and apoptosis are frequent cellular responses to persistent DNA damage. The failure of the PGC pool to expand between E8.5-E12.5 in TLS-deficient embryos could be explained by either reduced PGC proliferation, increased PGC death, or a combination of both. We therefore first asked if TLS-deficiency leads to increased PGC death by staining E12.5 urogenital ridges for the apoptotic marker cleaved-caspase 3 (CC3)<sup><xref ref-type="bibr" rid="CR85">85</xref></sup>. We observed significant 7.7-fold and 5.1-fold increases in the proportion of apoptotic (CC3<sup>+</sup>) PGCs in <italic>Pcna</italic><sup><italic>R/R</italic></sup> and <italic>Rev1</italic><sup><italic>-/-</italic></sup> embryos respectively (Fig. <xref ref-type="fig" rid="Fig4">4d</xref>). In contrast, there was no significant increase in the proportion of CC3<sup>+</sup> somatic cells in either mutant (Supplementary Fig. <xref ref-type="media" rid="MOESM1">8b</xref>). We next assessed in vivo PGC proliferation by injecting pregnant dams with a single dose of ethynyl-2’-deoxyuridine (EdU), which is incorporated into the DNA of replicating cells allowing the quantification of the proportion of cells in S-phase during the EdU pulse (Supplementary Fig. <xref ref-type="media" rid="MOESM1">8c</xref>)<sup><xref ref-type="bibr" rid="CR86">86</xref></sup>. We found a significant reduction in the frequency of EdU<sup>+</sup> PGCs in both <italic>Pcna</italic><sup><italic>R/R</italic></sup> and <italic>Rev1</italic><sup><italic>-/-</italic></sup> embryos compared to wildtype (Fig. <xref ref-type="fig" rid="Fig4">4e</xref>). In the soma however, we observed a comparable number of EdU<sup>+</sup> somatic cells in mutant and wildtype embryos, suggesting that the reduced proliferation is restricted to the germ cell compartment (Supplementary Fig. <xref ref-type="media" rid="MOESM1">8d</xref>). To assess if reduced incorporation of EdU may be due to defects in cell cycle kinetics, we assessed the cell cycle properties of PGCs. First, we stained cells for Ser-10 phosphorylation of histone H3 (pH3) that occurs during cell division facilitating chromatin compaction, necessary for mitosis<sup><xref ref-type="bibr" rid="CR87">87</xref></sup>. We found that a higher proportion of PGCs stained positive for pH3 in <italic>Pcna</italic><sup><italic>R/R</italic></sup> and <italic>Rev1</italic><sup><italic>-/-</italic></sup> genital ridges (Fig. <xref ref-type="fig" rid="Fig4">4f</xref>). We also looked at the localization of cyclin B1 which is in the cytoplasm during G2, before becoming phosphorylated during mitosis, driving its re-localization to the nucleus<sup><xref ref-type="bibr" rid="CR88">88</xref>,<xref ref-type="bibr" rid="CR89">89</xref></sup>. At E12.5 15.7% of wildtype PGCs had nuclear cyclin B1 and were therefore in G2/M-phase (Supplementary Fig. <xref ref-type="media" rid="MOESM1">8e</xref>). This was significantly higher in the absence of TLS with a 2.6-fold increase in G2/M-phase PGCs in <italic>Pcna</italic><sup><italic>R/R</italic></sup> gonads (Supplementary Fig. <xref ref-type="media" rid="MOESM1">8e</xref>). These data reveal that <italic>Pcna</italic><sup><italic>R/R</italic></sup> and <italic>Rev1</italic><sup><italic>-/-</italic></sup> PGCs accumulate markers of DNA damage, are less proliferative, accumulate at the G2/M phase of the cell cycle and are more likely to undergo programmed cell death. The combination of these cellular defects likely explains the profound reduction in the number of PGCs in the absence of TLS factors.</p>
</sec>
<sec id="Sec7">
<title>Pcna<sup>R/R</sup> and Rev1<sup>-/-</sup> PGCs are developmentally blocked</title>
<p id="Par19">Despite their scarcity, there are a few remaining PGCs in <italic>Pcna</italic><sup><italic>R/R</italic></sup> and <italic>Rev1</italic><sup><italic>-/-</italic></sup> embryos at E12.5 (medians <italic>Pcna</italic><sup><italic>R/R</italic></sup> = 7 and <italic>Rev1</italic><sup><italic>-/-</italic></sup> = 15). However, adult mice are completely sterile suggesting that the remaining PGCs are not able to generate functional gametes. Successful PGC development requires the activation of the germ cell transcriptional program coupled to global epigenetic changes<sup><xref ref-type="bibr" rid="CR90">90</xref>–<xref ref-type="bibr" rid="CR92">92</xref></sup>. We therefore set out to test if <italic>Pcna</italic><sup><italic>R/R</italic></sup> and <italic>Rev1</italic><sup><italic>-/-</italic></sup> PGCs underwent these critical transcriptional and epigenetic processes. Initially, we performed gene expression analysis on E12.5 PGCs and found that both <italic>Pcna</italic><sup><italic>R/R</italic></sup> and <italic>Rev1</italic><sup><italic>-/-</italic></sup> PGCs expressed the early markers of PGC development <italic>Nanos3</italic> and <italic>Prdm1</italic> at comparable levels to wildtype (Fig. <xref ref-type="fig" rid="Fig5">5a</xref>). Conversely, expression of the later-stage marker <italic>Mvh</italic> was dramatically reduced in both mutants (Fig. <xref ref-type="fig" rid="Fig5">5a</xref>). We extended the gene expression analysis to additional germ cell specific genes in <italic>Pcna</italic><sup><italic>R/R</italic></sup> E12.5 PGCs. Consistent with the gene expression data presented above, genes normally expressed in the later stages of PGC development (<italic>Dazl</italic>, <italic>Mili</italic>, <italic>Mael, Sycp3, Mov10l1, Hormad1</italic> and <italic>Brdt</italic>) had reduced expression compared to wildtype which was not true for early-stage genes (<italic>Stella</italic> and <italic>Fragilis</italic>) (Fig. <xref ref-type="fig" rid="Fig5">5b</xref>). Therefore, E12.5 TLS-deficient PGCs transcriptionally resemble earlier stages of development.<fig id="Fig5"><label>Fig. 5</label><caption><title>TLS preserves the PGC developmental programme.</title><p><bold>a</bold> RT-qPCR expression analysis of PGCs from E12.5 embryos (<italic>n</italic> = 3 embryos per genotype. Data represent mean and s.d. <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test). <bold>b</bold> RT-qPCR expression analysis of PGCs from E12.5 embryos. (Wildtype <italic>n</italic> = 6; <italic>Pcna</italic><sup><italic>R/R</italic></sup>
<italic>n</italic> = 7. Except for <italic>Hormad1</italic> and <italic>Brdt</italic>, wildtype <italic>n</italic> = 3; <italic>Pcna</italic><sup><italic>R/R</italic></sup>
<italic>n</italic> = 5). <bold>c</bold> Box plot of global DNA CpG methylation levels in E6.5 epiblast cells and E12.5 PGCs. The center displays the median, boxes the interquartile range and whiskers the minimum and maximum of CpG methylation distribution of the genome in 5Kbp genomic windows (<italic>n</italic> = 513027, 318836, 498368). <bold>d</bold> Circos-plot representation of DNA methylation levels in E6.5 epiblast and E12.5 wildtype PGCs and E12.5 <italic>Pcna</italic><sup><italic>R/R</italic></sup> PGCs. CpG methylation was averaged in 5 Mbp genomic windows and the average DNA methylation is represented as a histogram track. <bold>e</bold> Heatmap showing methylation levels for E6.5 epiblast and E12.5 PGCs. <bold>f</bold> Unclustered methylation heatmap of CpG methylation in E6.5 epiblast and E12.5 PGCs. <bold>g</bold> Bisulfite sequencing and quantification of the <italic>Line-1</italic> element from E12.5 wildtype, <italic>Pcna</italic><sup><italic>R/R</italic></sup> and <italic>Rev1</italic><sup><italic>-/-</italic></sup> embryos (filled:methylated CpG, open:unmethylated CpG. Each point represents one embryo, data represent mean and s.d., <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test). <bold>h</bold> Violin plots reflecting the DNA methylation levels of GRR gene bodies in E6.5 epiblast cells from wildtype embryos, E12.5 wildtype PGCs and E12.5 <italic>Pcna</italic><sup><italic>R/R</italic></sup> PGCs. The center displays the median, boxes the interquartile range and whiskers the minimum and maximum of percentage methylation calculated over each gene body with each point representing an individual gene (<italic>n</italic> = 37, 37, 35 left to right). <bold>i</bold> CpG methylation across selected GRR genes in E6.5 epiblast and E12.5 PGCs. The plots represent the distribution of CpG methylation across genes segmented in 0.1 Kbp genomic windows. <bold>j</bold> RT-qPCR expression analysis of <italic>Tet1</italic> and <italic>Dnmt1</italic> in PGCs from E12.5 embryos. (For <italic>Tet1</italic>, <italic>n</italic> = 5 for both genotypes. <italic>Dnmt1</italic>, wildtype <italic>n</italic> = 6 and <italic>Pcna</italic><sup><italic>R/R</italic></sup>
<italic>n</italic> = 7. Data represent mean and s.d., <italic>P</italic> values were calculated by a two-tailed Mann–Whitney <italic>U</italic>-test).</p></caption><graphic id="d33e2372" xlink:href="41467_2024_47219_Fig5_HTML"></graphic></fig></p>
<p id="Par20">The expression of germ cell specific genes in PGCs is activated during epigenetic reprogramming, specifically through DNA demethylation of gene promoters<sup><xref ref-type="bibr" rid="CR14">14</xref>,<xref ref-type="bibr" rid="CR90">90</xref>–<xref ref-type="bibr" rid="CR92">92</xref></sup>. The process of DNA demethylation occurs across the whole genome and is unique to the PGC compartment in the embryo. Alongside gene expression regulation, DNA demethylation is critical for imprint erasure and X-chromosome reactivation, processes needed for germ cell function. Our gene expression data prompted us to assess DNA methylation of PGCs. We performed whole genome bisulfite sequencing (WGBS) on wildtype and <italic>Pcna</italic><sup><italic>R/R</italic></sup> FACS-purified E12.5 PGCs and compared these to wildtype E6.5 epiblast cells, which are the origin of PGCs (Fig. <xref ref-type="fig" rid="Fig5">5c</xref>). Compared to epiblast cells, the genome of wildtype E12.5 PGCs had extremely low levels of DNA CpG methylation, reflecting demethylation during PGC development. In contrast, in <italic>Pcna</italic><sup><italic>R/R</italic></sup> PGCs we found a near complete retention of DNA CpG methylation (Fig. <xref ref-type="fig" rid="Fig5">5c</xref>). We mapped the methylation levels across the genome and found the retention in <italic>Pcna</italic><sup><italic>R/R</italic></sup> PGCs was genome-wide (Fig. <xref ref-type="fig" rid="Fig5">5d</xref>).</p>
<p id="Par21">DNA methylation does not proceed uniformly, with different genomic features reaching the lowest level of methylation at different times<sup><xref ref-type="bibr" rid="CR7">7</xref>,<xref ref-type="bibr" rid="CR14">14</xref>,<xref ref-type="bibr" rid="CR91">91</xref>,<xref ref-type="bibr" rid="CR92">92</xref></sup>. We therefore asked if only a subset of genomic features retained DNA methylation in TLS mutants or if the defect was truly global. Analysis of gene bodies and promoters showed that these were hypermethylated in <italic>Pcna</italic><sup><italic>R/R</italic></sup> PGCs compared to wildtype, suggesting a failure in the early, global phase of demethylation (Fig. <xref ref-type="fig" rid="Fig5">5e, f</xref>). We also found that repeat elements (LINE-1, SINE elements, and endogenous retroviruses) retained methylation in <italic>Pcna</italic><sup><italic>R/R</italic></sup> PGCs (Fig. <xref ref-type="fig" rid="Fig5">5f</xref> and Supplementary Fig. <xref ref-type="media" rid="MOESM1">9a</xref>). We validated this result by locus-specific, targeted bisulfite DNA sequencing of LINE-1 as it makes up ~20% of the genome and again found increased methylation in both <italic>Pcna</italic><sup><italic>R/R</italic></sup> and <italic>Rev1</italic><sup><italic>-/-</italic></sup> PGCs (Fig. <xref ref-type="fig" rid="Fig5">5g</xref> and Supplementary Fig. <xref ref-type="media" rid="MOESM1">10a−d</xref>). We went on to analyze features which undergo DNA demethylation in the later wave and for which demethylation is both characteristic of germ cell development but also required for function. We found that imprinted differentially methylated regions (DMRs) were hypermethylated in E12.5 <italic>Pcna</italic><sup><italic>R/R</italic></sup> PGCs, including both maternally and paternally methylated DMRs (Supplementary Fig. <xref ref-type="media" rid="MOESM1">9b</xref>). Thus, this analysis reveals that the later wave of DNA demethylation also fails in the absence of TLS.</p>
<p id="Par22">CGIs (CpG islands) in the promoters of genes associated with germ cell specific processes have been shown to retain DNA methylation until E11.5 and be demethylated in the later wave of PGC DNA demethylation<sup><xref ref-type="bibr" rid="CR14">14</xref></sup>. These genes are involved in meiosis and gamete generation and are only expressed in germ cells but silenced and methylated in somatic cells. We found that these promoters were hypermethylated in mutant PGCs (Supplementary Fig. <xref ref-type="media" rid="MOESM1">9c</xref>). We next analyzed CpG methylation of genes required for germ cell production whose transcriptional activation is concurrent with promoter demethylation during the later stages of PGC development (germline reprogramming responsive genes - GRR genes)<sup><xref ref-type="bibr" rid="CR92">92</xref></sup>. CpG methylation analysis of a panel of 45 GRRs showed that these were largely demethylated in E12.5 wildtype PGCs (Fig. <xref ref-type="fig" rid="Fig5">5h</xref>). In <italic>Pcna</italic><sup><italic>R/R</italic></sup> E12.5 PGCs however, these were hypermethylated. We confirmed this by performing locus-specific BS-Seq of the promoters of two GRR genes <italic>Mili</italic> and <italic>Dazl</italic> (Supplementary Fig. <xref ref-type="media" rid="MOESM1">10e, f</xref>). It is striking that we observed both reduced expression of the GRR genes <italic>Mvh</italic>, <italic>Dazl</italic>, <italic>Mili</italic>, <italic>Mael, Sycp3, Mov10l1, Hormad1</italic> and <italic>Brdt</italic> and also hypermethylation of these loci in <italic>Pcna</italic><sup><italic>R/R</italic></sup> PGCs (Fig. <xref ref-type="fig" rid="Fig5">5b</xref>, <xref ref-type="fig" rid="Fig5">i</xref> and Supplementary Fig. <xref ref-type="media" rid="MOESM1">9d</xref>). This retention of DNA methylation in TLS mutants may explain the reduced activation of GRRs</p>
<p id="Par23">We compared the levels of DNA CpG methylation in mutant and wildtype E12.5 PGCs to the E6.5 epiblast. The retention of methylation across a panel of genomic features revealed that E12.5 <italic>Pcna</italic><sup><italic>R/R</italic></sup> PGCs were more akin to the epiblast than to wildtype PGCs (Fig. <xref ref-type="fig" rid="Fig5">5e−g</xref>). Together, these data show that loss of TLS prevents global DNA demethylation in PGCs with retention occurring at early and late demethylated regions. Furthermore, E12.5 mutant PGCs appear trapped at an earlier stage of development with a methylome similar to E6.5 epiblast cells.</p>
<p id="Par24">Next, we focused on understanding the basis for methylation retention in the absence of TLS and its effect on expression of germline genes. First, we studied the machinery that normally adds methylation marks to DNA by either ensuring methylation patterns are inherited during cell division or by depositing new methylation marks. Following DNA replication, the newly synthesized DNA strand is unmethylated and DNMT1 is recruited to hemimethylated CpG dyads and transfers a methyl group to the unmethylated cytosine whilst DNMT3A and DNMT3B adds methylation marks to unmethylated DNA to deposit de novo methylation. We measured the expression of <italic>Dnmt1</italic>, <italic>Dnmt3a</italic> and <italic>Dnmt3b</italic> and found no difference between wildtype and <italic>Pcna</italic><sup><italic>R/R</italic></sup> E12.5 PGCs suggesting that overexpression of the methyltransferases is not responsible for DNA hypermethylation in the absence of TLS (Fig. <xref ref-type="fig" rid="Fig5">5j</xref> and Supplementary Fig. <xref ref-type="media" rid="MOESM1">9e</xref>). The enzyme TET1 has been shown to play important DNA demethylation dependent and independent roles in regulating the expression of genes critical for gamete formation. Notably, TET1 has previously been shown necessary for the transcriptional activation and maintenance of low levels of methylation at GRRs<sup><xref ref-type="bibr" rid="CR92">92</xref></sup>. We quantified the expression of <italic>Tet1</italic> and its family member <italic>Tet2</italic> in wildtype and <italic>Pcna</italic><sup><italic>R/R</italic></sup> E12.5 PGCs and found no difference indicating that failure to activate the GRR genes is not mediated through <italic>Tet1</italic> expression (Fig. <xref ref-type="fig" rid="Fig5">5j</xref> and Supplementary Fig. <xref ref-type="media" rid="MOESM1">9f</xref>). We further validated this through TET1 immunostaining in E12.5 urogenital ridges which revealed the presence of protein in wildtype and <italic>Pcna</italic><sup><italic>R/R</italic></sup> E12.5 PGCs (Supplementary Fig. <xref ref-type="media" rid="MOESM1">9g</xref>).</p>
<p id="Par25">Together, these data reveal that E12.5 TLS-deficient PGCs resemble earlier stages of germ cell development transcriptionally and in their methylome. As DNA methylation regulates gene expression it is plausible that the retention observed in TLS mutants may contribute to the observed reduction in expression of germ cell genes. As well as failure of GRR demethylation, mutant PGCs also fail to erase imprinted DMR methylation, processes essential for normal germ cell function.</p>
</sec>
</sec>
<sec id="Sec8" sec-type="discussion">
<title>Discussion</title>
<p id="Par26">Declining birth rates and increasing infertility are drawing attention to environmental and genetic causes of human infertility. GWAS studies have identified the DDR as an important regulator of this, however mechanistic studies have for the most part been lacking. Our current work reveals a crucial role for DNA translesion synthesis during germline development. Our findings demonstrate that REV1 is required for both human PGCLC and mouse PGC development, indicating a conserved embryonic cause of infertility.</p>
<p id="Par27">Alongside REV1, we show that REV7 and the modification site of PCNA, K164, are essential for embryonic germ cell development. Though all three of these factors can act in multiple different processes, our results assert that it is their common role in TLS that is required<sup><xref ref-type="bibr" rid="CR50">50</xref></sup>. REV1 has both catalytic and non-catalytic functions and we have found that the C-terminal protein interaction domain, critical for TLS, is required in PGCs. Like REV1, the monoubiquitination of PCNA at K164 serves as a TLS scaffold. REV7 interacts with REV3L to form the TLS extender polymerase called Polζ which has a specialized function in catalyzing DNA synthesis after lesion bypass during TLS. Importantly, Polζ is recruited to TLS sites through direct interactions between its REV7 subunit and REV1’s C-terminal domain<sup><xref ref-type="bibr" rid="CR50">50</xref>,<xref ref-type="bibr" rid="CR57">57</xref></sup>. Our finding that both the REV1 C-terminus and REV7 are essential in PGCs supports a role for both in TLS. Our results did not identify an inserter TLS polymerase which may be explained by the well-characterized redundancy between the inserter polymerases<sup><xref ref-type="bibr" rid="CR18">18</xref>,<xref ref-type="bibr" rid="CR19">19</xref></sup>. We found that the three core TLS factors REV1, REV7 and PCNA K164 are required during embryonic stages of germ cell development and have identical timing and magnitude of PGC defects. In addition, the remaining PGCs in <italic>Rev1</italic><sup><italic>-/-</italic></sup> and <italic>Pcna</italic><sup><italic>R/R</italic></sup> embryos are phenotypically indistinguishable. This striking phenotypic overlap suggests that the cause of defect is the same in all three mutants with TLS being the only function common to all three factors. Whilst REV7 has described functions in DNA DSB repair and mitosis our data suggest that it is the role of REV7 in Polζ that is required for normal PGC development. Together, these data argue that TLS plays an essential role in fertility by preserving the development of PGCs.</p>
<p id="Par28">DNA repair pathways such as Fanconi anemia (FA) DNA crosslink repair and DNA DSB repair, have previously been shown to be necessary for PGC development<sup><xref ref-type="bibr" rid="CR30">30</xref>,<xref ref-type="bibr" rid="CR32">32</xref>,<xref ref-type="bibr" rid="CR36">36</xref>–<xref ref-type="bibr" rid="CR38">38</xref></sup> (23, 25, 70−72). It is interesting that both genetic and biochemical studies have shown that the TLS factors REV1 and Polζ (REV7-REV3L) act in a common pathway with the FA repair proteins to maintain cellular resistance to DNA interstrand crosslinking agents<sup><xref ref-type="bibr" rid="CR93">93</xref>–<xref ref-type="bibr" rid="CR96">96</xref></sup>. Indeed we found that <italic>Pcna</italic><sup><italic>R/R</italic></sup><italic>, Rev1</italic><sup><italic>-/-</italic></sup>
<italic>and Rev7</italic><sup><italic>-/-</italic></sup> MEFs were mildly hypersensitive to DNA crosslinking agents, however this sensitivity was much less than in the absence of FANCA (Supplementary Fig. <xref ref-type="media" rid="MOESM1">11a</xref>). This does however leave the possibility that the loss of PGCs in FA-deficient and TLS-deficient mice shares a common cause. However, the magnitude and the temporality of the defects are different between the two classes of mutants with the loss of PGCs in TLS-deficient mice occurring 48 h before a loss of PGCs is observed in FA-mutants<sup><xref ref-type="bibr" rid="CR30">30</xref></sup>. The magnitude of PGC loss in E12.5 TLS-deficient embryos is also two orders of magnitude greater than in the absence of the FA pathway<sup><xref ref-type="bibr" rid="CR30">30</xref></sup>. Furthermore, there are substantial phenotypic differences between TLS-deficient and FA-deficient PGCs - the loss of FA repair factors does not alter the expression of GRR genes nor block DNA demethylation unlike the loss of TLS components<sup><xref ref-type="bibr" rid="CR30">30</xref></sup>. This shows that whilst multiple DNA repair pathways are required for normal PGC development, they have distinct roles with different phenotypic outcomes.</p>
<p id="Par29">The phenotypic features of the few remaining PGCs in TLS-deficient embryos explains the basis of germ cell loss and sterility. We found that mutant PGCs have a higher burden of DNA damage, apoptosis and cell cycle abnormalities as well as a failure in DNA demethylation and activation of the germ cell transcriptional program. Together these phenotypic features are likely to explain the basis of the failure of PGC development and infertility. Our results reveal that TLS is essential for the developmental processes that are needed for the production of mature germ cells<sup><xref ref-type="bibr" rid="CR4">4</xref>,<xref ref-type="bibr" rid="CR92">92</xref></sup>.</p>
<p id="Par30">The increase in DNA damage markers, whilst not previously shown in a physiological context, are consistent with known consequences of TLS deficiency upon genotoxin exposure in tissue culture systems. We observe an increase in the frequency of TLS-deficient PGCs with ɣ-H2A.X foci, RPA32 foci and CHK2 phosphorylation. Whilst none of these markers are entirely specific to the accumulation of DNA DSBs the combination of all three strongly suggests that in the absence of TLS PGCs accumulate DNA DSBs<sup><xref ref-type="bibr" rid="CR84">84</xref></sup>. We also observed that an increased proportion of TLS-deficient PGCs are pH3-positive and have nuclear cyclin B1 localization, and therefore are at the G2/M phase of the cell cycle, with a reduced number staining positive for EdU following a 4 h pulse showing that fewer replicate their DNA and hence go through S-phase. These findings are consistent with a role for TLS either during S-phase (on-the-fly TLS) or in post-replicative repair. Whilst this data does not tell us when during the cell cycle DNA damage occurs it does suggest that it poses a block to mitosis following DNA synthesis unless it is resolved by TLS. It is interesting to note that embryonic stem cells have a less effective G1 checkpoint<sup><xref ref-type="bibr" rid="CR97">97</xref>–<xref ref-type="bibr" rid="CR100">100</xref></sup>. As this appears to also be the case in PGCs it would suggest that cells can enter into S-phase laden with DNA damage thus necessitating replication coupled repair such as TLS for resolution of damage and entry into mitosis<sup><xref ref-type="bibr" rid="CR101">101</xref></sup>. Finally, we observed an increase in the proportion of TLS-deficient PGCs that undergo apoptosis which is a frequent cellular response to DNA damage. However, several studies have reported that loss of the DDR leads to a reduction in PGC number by altering the cell cycle rather than through apoptosis<sup><xref ref-type="bibr" rid="CR102">102</xref>,<xref ref-type="bibr" rid="CR103">103</xref></sup>. This difference may be due to the different repair pathways in these studies counteracting distinct classes of DNA damage that, when unrepaired, result in different cellular outcomes. However, apoptosis does play a role in the clearance of PGCs during unperturbed development or when epigenetic regulators, that normally suppress transposons, are lost<sup><xref ref-type="bibr" rid="CR104">104</xref>–<xref ref-type="bibr" rid="CR106">106</xref></sup>. Taken together, our data shows that in the absence of TLS PGCs accumulate markers of DNA damage, induce cell cycle arrest at the G2/M phase of the cell cycle and undergo apoptosis. It is likely that a combination of cell cycle arrest and apoptosis ultimately results in the numerical PGC defect. Whilst these findings are consistent with the described role of TLS factors in dealing with DNA damage, the failure to undergo DNA demethylation or transcribe germ cell factors is entirely unexpected.</p>
<p id="Par31">Whilst wildtype PGCs have very low levels of CpG methylation by E12.5, we found that TLS-deficient PGCs have genome-wide retention of DNA methylation. Indeed, we found that the methylation of TLS-deficient PGCs at E12.5 was similar to that seen in E6.5 epiblast cells. These data argue that the PGC precursors in the epiblast have undergone de novo methylation during early embryogenesis before later undergoing demethylation in PGC development. The observed cell cycle defects could explain why demethylation is prevented in the absence of TLS. Cell cycle arrested PGCs may be unable to respond to time-dependent cell extrinsic cues, such as signaling molecules that instruct PGCs to proceed in their development. Alternatively, the cell cycle perturbations may block a cell-intrinsic process, preventing activation of the germ cell program and epigenetic reprogramming. Interestingly, the first stage of global DNA demethylation in PGCs relies upon replication to dilute methylation marks<sup><xref ref-type="bibr" rid="CR14">14</xref>,<xref ref-type="bibr" rid="CR92">92</xref></sup>. Therefore, the cell cycle arrest in TLS-deficient PGCs may result in the genome-wide retention of methylation marks that we observe. The normal expression of TET1 in mutant PGCs argues against dysregulation of either its enzymatic or non-enzymatic functions as a contributing factor to the phenotype observed.</p>
<p id="Par32">Whilst early stage PGC genes are induced by extrinsic signals and maintained by PGC-specific and pluripotency-associated transcription factors, DNA demethylation is important for the transcription of genes expressed in the later stages of PGC development. Therefore, the failure to transcribe GRR genes and retention of DNA methylation may not be independent of each other. Indeed, we find reduced mRNA expression and DNA hypermethylation of each GRR gene that was analyzed. It is therefore likely that the block in cell cycle progression in the absence of TLS prevents DNA demethylation, transcription of germ cell factors and therefore blocks PGC development.</p>
<p id="Par33">The specificity of the phenotype which only affects germ cell homeostasis is intriguing. Despite loss of genome stability in the erythroid lineage and a reduction in blood stem cell frequency that we show begins in utero, TLS-deficient mice sustain peripheral blood homeostasis throughout life. Moreover, in vivo assessment of rapidly dividing tissues in <italic>Pcna</italic><sup><italic>R/R</italic></sup> mice revealed no defects or overt phenotypes in somatic tissue homeostasis and consistent with this, mutant mice show normal longevity (Supplementary Figs. <xref ref-type="media" rid="MOESM1">5</xref> and <xref ref-type="media" rid="MOESM1">6</xref>). In contrast, there is a complete failure to generate functional gametes. This striking dichotomy raises the question of why TLS is only required to maintain homeostasis in the germ cell compartment. We have demonstrated that TLS is required for processes that occur only in the germ cell lineage: engagement of germline genes and genome-wide DNA demethylation. In addition, we find that the canonical phenotypes associated with TLS deficiency and the DNA damage response are also restricted to the germ cell compartment: unresolved DNA damage, apoptosis, and cell cycle perturbation (Fig. <xref ref-type="fig" rid="Fig4">4</xref> and Supplementary Fig. <xref ref-type="media" rid="MOESM1">8</xref>). This raises the possibilities that (i) the DNA damage checkpoints that activate apoptosis or cell cycle arrest are different in PGCs than in somatic cells, (ii) there are replication impediments or other features unique to PGCs that confer a dependency on TLS activity or (iii) that PGCs are more reliant upon TLS to resolve replication impediments than somatic cell types.</p>
<p id="Par34">Future work will be required to determine which of these possibilities explains the selective requirement for TLS during PGC development. Recent work has shown that PGCs frequently experience transcription-replication conflicts and it is plausible that TLS factors are required to overcome such sources of genome instability<sup><xref ref-type="bibr" rid="CR107">107</xref>,<xref ref-type="bibr" rid="CR108">108</xref></sup>. Alternatively, the lack of a G1 cell cycle checkpoint in PGCs may render these cells uniquely dependent upon TLS to bypass damage during S-phase that in other cells would be repaired prior to the initiation of replication. Finally, it is tempting to speculate that the dependency of PGC development on TLS may be directly connected to early epigenetic reprogramming events that are required for subsequent germline development.</p>
</sec>
<sec id="Sec9">
<title>Methods</title>
<sec id="Sec10">
<title>Human iPSC culture</title>
<p id="Par35">The hiPSC cell line (BTAG 585b1-868) was a kind gift from Prof. Mitinori Saitou (Institute for the Advanced Study of Human Biology, Kyoto, Japan). Cells were maintained in Stemfit Basic04 (Amsbio, SFB-504-CT) on Geltrex (Life Technologies, A1413302) coated dishes. Cells were passaged as single cells using Accutase (Millipore, SF006) and 10 μM of rock inhibitor (Y-27632, STEMCELL Technologies, 72304) was added for the first 24 h after seeding. Cells were maintained in hypoxic conditions (5% CO<sub>2</sub>, 5% O<sub>2</sub>) and without antibiotics.</p>
</sec>
<sec id="Sec11">
<title>Induction of iMeLCs and hPGCLCs</title>
<p id="Par36">Induction of iMeLCs and hPGCLCs was performed as previously published by Sasaki et al. (26). For iMeLC induction, hiPSCs were dissociated as single cells using a 1:1 mixture of TrypLE (Life Technologies, 12604013) and EDTA/PBS. 1.5 × 10<sup>5</sup> cells were seeded on a fibronectin coated well of a 12 well plate in GK15 medium (GMEM [Life Technologies] with 15% KSR, 0.1 mM NEAA, 2 mM L-glutamine, 1 mM sodium pyruvate, and 0.1 mM 2-mercaptoethanol) supplemented with: 50 ng/μl of ActivinA (R&amp;D, 338-AC-010), 3 μM of CHIR99021 (Cambridge Bioscience, SM13-5) and 10 μM of Y-27632. For hPGCLCs, 3.0 × 10<sup>3</sup> iMeLCs were plated into ulta-low attachment U-bottom 96-well plate (PHCBI, MS-9096UZ) in GK15 supplemented with: 10 ng/ml LIF (Millipore, LIF1005), 200 ng/ml of BMP4 314-BP-010, 100 ng/ml of SCF (R&amp;D Systems, 255-SC-050), 50 ng/ml of EGF (R&amp;D Systems, 236-EG-200), and 10 μM of Y-27632.</p>
</sec>
<sec id="Sec12">
<title>Generation of <italic>Rev1</italic><sup><italic>-/-</italic></sup> BTAG hiPSC line</title>
<p id="Par37">Two gRNAs were designed to target the exon 3 of <italic>REV1</italic> (Supplementary Table <xref ref-type="media" rid="MOESM1">2</xref>) and subcloned into the PX458 plasmid (Addgene, 48138). 5.0−8.0 × 10<sup>5</sup> BTAG cells were electroporated with 3 μg of each gRNA using Amaxa 4D nucleofector (Lonza, V4XP-3012) and the CA-137 program. 48 h after the transfection, GFP positive cells were FACS sorted and plated in 96 well plate with one cell per well in Stemfit Basic04, 10 μM of Y-27632, penicillin and streptomycin (Life Technologies, 15070063). After 10−14 days single cell-derived colonies were picked and expanded. Clones were screened by PCR on extracted genomic DNA (NEB, T3010L) using the primer pair listed in Supplementary Table <xref ref-type="media" rid="MOESM1">2</xref>.</p>
</sec>
<sec id="Sec13">
<title>Mitomycin C (MMC) treatment of BTAG hiPSC cells</title>
<p id="Par38">Wildtype and <italic>Rev1</italic><sup><italic>-/-</italic></sup> cells were passaged as single cells then seeded at 1:10 density in Stemfit Basic04 and Y-27632. 24 h after, medium was changed and Y-27632 was removed. The next day, 5 ng/ml MMC (Insight Biotechnology, sc-3514A) was added to the medium and 48−72 h later wells were assessed for surviving hiPSC colonies.</p>
</sec>
<sec id="Sec14">
<title>Embryoid body formation assay</title>
<p id="Par39">hiPSCs at 70−80% confluency were dissociated to a single cell suspension with Accutase. 2.4 × 10<sup>^6</sup> cells were seeded in an AggreWell<sup>TM</sup>400 (Stemcell technologies) plate in embryoid body (EB) formation medium (Stemcell technologies, 05893) supplemented with Y-27632 rock inhibitor and incubated for two days at 37 °C with 5% CO<sub>2</sub>. Newly formed EBs were collected from the wells and centrifuged at 160 x <italic>g</italic> for 3 min, resuspended in EB medium: DMEM/F-12 with Glutamax (ThermoFisher, 10565-018), 20% KSR (ThermoFisher, 10828010), 1x NEAA (ThermoFisher, 11140-050) and 2-Mercaptoethanol (ThermoFisher, 21985-023) then plated in 10 cm dishes and maintained at 37 °C with 5% CO<sub>2</sub>. Medium was changed every other day for 21 days. EBs were collected, washed, trypsinised and used for RT-qPCR analysis.</p>
</sec>
<sec id="Sec15">
<title>Embyroid body gene expression analysis</title>
<p id="Par40">Total RNA was extracted from hiPSCs and their counterpart day 21 EBs using the RNeasy kit (Qiagen) according to the manufacturer’s instructions. First-strand complementary DNA was synthesized using PrimeScript RT Reagent Kit (Takara) according to the manufacturer’s instructions. PCR amplification was performed using the SensiMix SYBR No-ROX Kit (Bioline) on a CFX96 machine (Biorad). The primers used for this experiment are listed in Supplementary Table <xref ref-type="media" rid="MOESM1">3</xref>. Mean threshold values were determined using standard comparative C<sub>T</sub> methods. All expression levels were normalized to the hiPSC line used to generate the EBs.</p>
</sec>
<sec id="Sec16">
<title>Proliferation assays</title>
<p id="Par41">hiPSCs were plated into flat-bottomed 96-well plates (ThermoFisher, 167008) at a density of 1000 cells per well in Stemfit supplemented with Y-27632 rock inhibitor. Every day for 5 days the absorbance of the different samples was assessed. Briefly 20 µl of MTS reagent (Promega, G3582) was added to each well and the plate was incubated at 37 °C for two hours. Following incubation, absorbance was measured at 490 nm using a plate reader.</p>
</sec>
<sec id="Sec17">
<title>hPGCLC immunostaining</title>
<p id="Par42">Day 4 hPGCLC aggregates were fixed in 4% PFA for 20 min at room temperature. After PBS washes and incubation in 30% sucrose, aggregates were embedded in OCT (VWR, 361603E) and sectioned at a thickness of 10 μm. Samples were permeabilized for 20 min in PBS/0.2% Triton X100 (PBST) and blocked for 1 h in blocking solution (Santa Cruz, sc-516214). Samples were then incubated with the following primary antibodies diluted in blocking buffer at room temperature for 2 h: anti-OCT4 (1:200, catalog no. ab181557; Abcam); anti-TFAP2C (1:100, catalog no. sc-12762; Santa Cruz); anti-SOX17 (1:200, catalog no. AF1924; R&amp;D). After PBS washes, samples were incubated with the following secondary antibodies diluted 1:1000 in PBS: Donkey anti-rabbit IgG (Alexa Fluor 594, catalog no. A21207; Invitrogen); Donkey Anti-Goat IgG H&amp;L (Alexa Fluor 488, catalog no. ab150129; Abcam); Donkey anti-Mouse IgG (H&amp;L) (Alexa fluor 594, catalog no. R37115; Life Technologies). Sections were then washed with PBS and incubated for 10 min with DAPI. A coverslip was then placed on the slides in Vectashield vibrance (Vector Labs, H-1800-10). Sections were imaged using the SP5 inverted confocal microscope (Leica).</p>
</sec>
<sec id="Sec18">
<title>Flow cytometry analysis of hPGCLC aggregates</title>
<p id="Par43">Day 4 aggregates were collected and dissociated with 0.25% trypsin-EDTA (Life Technologies, 25200056) for 10 min at 37 °C under agitation. Cells were washed with PBS containing fetal bovine serum (FBS) and 0.1% bovine serum albumin (BSA) before being subjected to centrifugation. Dissociated cells were then resuspended in FACS buffer (PBS, 0.1% BSA), filtered by a cell strainer (BD Biosciences) and analysed or sorted by FACS (Aria III, BD Biosciences) based on the eGFP and tdTomato reporters. Analysis of hPGCLCs differentiation efficiency was performed on one wild type clone and three REV1 KO clones in at least three independent experiments. FlowJo version 10 was used for analysis.</p>
</sec>
<sec id="Sec19">
<title>Mice</title>
<p id="Par44">All animal experiments performed in this study were approved by the Medical Research Council’s Laboratory of Molecular Biology animal welfare and ethical review body and conform to the UK Home Office Animal (Scientific Procedures) Act 1986 (License no PP6752216). All mice were maintained under specific pathogen-free conditions in independently ventilated cages (GM500; Techniplast) on Lignocel FS-14 spruce bedding (IPS) and provided with environmental enrichment (fun tunnel, chew stick and Enviro-Dri nesting material (LBS)) at 19−23 °C. Mice fed Dietex CRM pellets (SpecialDietService) ad libitum with light from 7:00 a.m. to 7:00 p.m. No mice used in this study were wild and no field-collected samples were used. All mice were maintained on an isogenic C57BL/6J background. Sex-specific analysis was performed for analysis of adult reproductive tissues as detailed in the text. Primordial germ cell analyses were performed before sexually divergent development of the germline and hence data from both sexes was combined. The lack of a sex-specific phenotype was confirmed through disaggregation and separate analysis of E12.5 PGC numbers for <italic>Pcna</italic><sup><italic>R/R</italic></sup> mutants. Embryos were examined at various developmental stages from E8.5 to E13.5 as indicated in the text. Female mice used in timed-mating experiments were aged between 6-25 weeks. The investigators were blinded to the genotypes of all mice throughout the study and data were acquired by relying entirely on identification numbers. The <italic>Stella-GFP</italic> (<italic>Tg(Dppa3/EGFP)6-25Masu</italic>) allele (MGI ID: 5519126) was a kind gift from Azim Surani<sup><xref ref-type="bibr" rid="CR77">77</xref></sup>. B6;129P2-<italic>Polk</italic><sup><italic>tm1.1Rsky</italic></sup><italic>/J</italic> allele (MGI ID: 2445458) and the B6.Cg-<italic>Polq</italic><sup><italic>tm1Jcs</italic></sup><italic>/J</italic> allele (MGI ID: 2155399) have been described previously<sup><xref ref-type="bibr" rid="CR60">60</xref>,<xref ref-type="bibr" rid="CR61">61</xref></sup>. <italic>GOF18</italic>-<italic>GFP</italic>(<italic>Tg(Pou5f1-EGFP)2Mnn</italic>) (MGI ID: 3057158) JAX (stock ID: 004654) mice were purchased from The Jackson Laboratory<sup><xref ref-type="bibr" rid="CR55">55</xref>,<xref ref-type="bibr" rid="CR56">56</xref></sup>. We imported the previously described C57BL/6NTac-Mad2l2&lt;tm1a(EUCOMM)Wtsi &gt; /WtsiCnbc strain (EM: 05374), frozen sperm from The European Mouse Mutant Archive and used to derive live mice<sup><xref ref-type="bibr" rid="CR45">45</xref></sup>.These mice were maintained on a C57BL/6J background.</p>
</sec>
<sec id="Sec20">
<title>Generation of <italic>Pcna</italic><sup><italic>K164R</italic></sup>, <italic>Rev1</italic><sup><italic>AA</italic></sup> and <italic>Rev1</italic><sup><italic>CT</italic></sup> mutant mice</title>
<p id="Par45">Mice carrying the <italic>Pcna</italic><sup><italic>K164R</italic></sup> and <italic>Rev1</italic><sup><italic>CT</italic></sup> alleles were generated by Alt-R CRISPR-Cas9 (IDT) mediated genome editing in zygotes on a C57BL/6J background. tracrRNA and crRNA were diluted to a final concentration of 1 μg/ul in injection buffer (10 mM Tris HCl pH 7.5, 0.1 mM EDTA). 5 μg of crRNA and 10 μg tracrRNA were mixed and annealed by heating to 95 °C for 5 min and then ramped to 25 °C at a rate of 5 °C per min. Alt-R SpCas9-3NLS was diluted to a concentration of 200 ng/μl in injection buffer. The RNP was assembled by diluting Alt-R SpCas9-3NLS and the annealed crRNA:tracrRNA at a final concentration of 20 ng/μl and incubated at room temperature for 15 min. The ssODN was then added to the RNP complex at a final concentration of 20 ng/μl and injected into zygotes. The zygotes were surgically transplanted into pseudopregnant CD1 females. Progeny were screened for correct gene targeting and the targeted allele sequenced. <italic>Rev1</italic><sup><italic>AA</italic></sup> mice were generated by gene targeting in mouse embryonic stem cells. Stella-GFP Bac9 ESCs were transfected with an ssODN and px451 plasmid containing a guide targeting exon 10 of <italic>Rev1</italic>. Clones were screened by PCR followed by AciI restriction digest. Two positive ESC clones were injected into C57BL6/J:TYR blastocysts. Mice with high levels of chimerism were back crossed and the progeny screened by PCR.</p>
</sec>
<sec id="Sec21">
<title>Isolation of mouse embryonic fibroblasts</title>
<p id="Par46">Timed matings were performed between heterozygous mice. Pregnant females were culled at E12.5 to harvest embryos. Embryos were incubated in pre-warmed trypsin solution (2.5 μg.mL<sup>−1</sup> trypsin (Gibco), 25 mM Tris, 120 mM NaCl, 25 mM KCl, 25 mM KH<sub>2</sub>PO<sub>4</sub>, 25 mM glucose, 25 mM EDTA, pH 7.6) for 10 min and disaggregated by gentle pipetting. Primary mouse embryonic fibroblast (MEF) cultures were established following standard methods and immortalized using the SV40 large T antigen as described previously. Briefly, Platinum-E retroviral packaging cells (Cell Biolabs) were transfected with pBABE-SV40-Puro and the culture media containing the virus was harvested 48 h later and passed through a 0.22 μm filter. The filtered retrovirus was mixed 1:1 with complete MEF media supplemented with 1 μg.mL<sup>−1</sup> hexadimethrine bromide (Polybrene, Millipore). The infective medium was subsequently added to primary MEF cultures and transformed clones were selected for 14 days using 1 μg.mL<sup>−1</sup> puromycin.</p>
</sec>
<sec id="Sec22">
<title>Measuring sensitivity of MEFs to DNA damaging agents</title>
<p id="Par47">Sensitivity to DNA-damaging agents was determined by seeding 1000 transformed MEFs per well of a 96-well flat-bottom plate and exposing to ultraviolet irradiation or mitomycin C (MMC). After 7 days of culture the MTS cell viability reagent (CellTiter 96® Aqueous One Solution Cell Proliferation Assay, Promega) was added and plates incubated for 4 h at 37 °C; absorbance was then measured at 492 nm.</p>
</sec>
<sec id="Sec23">
<title>Histological analysis</title>
<p id="Par48">Tissues were fixed in 10% neutral-buffered formalin for 24−36 h then transferred to 70% ethanol. Fixed samples were dehydrated and embedded in in paraffin and 4 μm sections cut. Sections were deparaffinised, re-hydrated and stained with haematoxylin and eosin (H&amp;E) following standard methods. Images were captured with an Eclipse Ti2-E (Nikon) microscope and tissue architecture was scored blindly.</p>
</sec>
<sec id="Sec24">
<title>Immunohistochemistry</title>
<p id="Par49">Formalin-fixed, paraffin-embedded samples were sectioned at 4 μm, deparaffinised and rehydrated following standard methods. Slides were boiled in antigen retrieval buffer (10 mM sodium citrate, pH 6) for 10 min and allowed to cool to room temperature before being washed three times in water for 5 min and then once in TBS, 0.1% w/v Tween-20 for 5 min. A hydrophobic ring was drawn around the tissue sections and samples incubated with blocking buffer (TBS, 0.1% w/v Tween-20, 5% v/v goat serum) for 1 h at room temperature. Samples were incubated with the following primary antibodies diluted in blocking buffer at 4 °C overnight: anti-PLZF (1:200, sc-28319) and anti-Ki67 (1:200, ab16667). Slides were washed three times with TBS, 0.1% w/v Tween-20 for 5 min and incubated with the following secondary antibodies for 1 hour at room temperature: swine anti-rabbit (1:200, catalog no. P0339, Dako) or anti-mouse horseradish peroxidase (HRP)-conjugated immunoglobulins (1:200, catalog no. P0339, Dako). For HRP-based immunohistochemistry, slides were incubated for 3−10 min with SignalStain diaminobenzidine substrate kit (catalogue no. 8059 P; Cell Signaling Technology) and then washed once in water for 5 min. Slides were dehydrated in an ethanol gradient following standard methods and finally in xylene before being mounted with DPX neutral mounting medium (catalog no. 317616, Sigma-Aldrich) and coverslips place on top of the slides. Images were captured with an Eclipse Ti2-E (Nikon) microscope. The frequency of PLZF<sup>+</sup> or Ki67<sup>+</sup> cells were scored blindly.</p>
</sec>
<sec id="Sec25">
<title>Assessing fertility of mice</title>
<p id="Par50">Mice were paired with wildtype C57BL/6J mice of the opposite sex. Female mice were monitored daily for the presence of copulation plugs and the number of offspring born over three successive months was recorded (only data from breeding pairs where at least 3 copulation plugs were observed was included in the analysis). Investigators performing the copulation plug checks were blinded to the genotypes of the mice.</p>
</sec>
<sec id="Sec26">
<title>Timed matings for embryo isolation</title>
<p id="Par51">Timed matings were performed overnight and female mice were assessed for the presence of copulation plugs the following day and separated from males. Halfway through the light cycle on the day a copulation plug was observed was designated E0.5. Pregnant mice were culled at noon of the appropriate day during gestation (E8.5–13.5) and the embryos harvested. Samples were processed immediately for further analysis with a small tissue biopsy taken for genotyping.</p>
</sec>
<sec id="Sec27">
<title>Immunofluorescence</title>
<p id="Par52">Timed matings were performed as described above and E12.5 embryos harvested. The fetal gonads were dissected out and fixed in PBS, 4% w/v paraformaldehyde for 30 min at 4 °C. Fixed samples were washed three times in PBS, 1% w/v Triton X-100 for 5 min at room temperature for permeabilization then pressed onto glass slides. A large hydrophobic ring was drawn around each sample and then incubated in blocking buffer (PBS, 1% w/v Triton X-100, 1% w/v BSA) for 1 h at room temperature. Samples were then incubated with the following primary antibodies at 4 °C overnight: anti-GFP (1:500, catalog no. GF090Rl; Nacalai); anti-GFP (1:2000, catalog no. ab13970); anti-phospho-Histone H2A.X (Ser139) (1:1000, catalog no. 05-636; Millipore); anti-cleaved caspase 3 (1:400, catalog no. 9661; Cell Signaling Technology); anti-MVH (ab27591); anti-RPA (1:100, catalog no, 2208; Cell Signaling Technology); anti-cyclin B1 (1:100, catalog no. 4138; Cell Signaling Technology); anti-phospho-Chk2 (Thr68) (1:100, catalog no, 2661; Cell Signaling Technology); anti-TET1 (1:100, ab272901) and anti-phospho-histone 3 (1:200, catalog no, 9701; Cell Signaling Technology). Samples were then washed three times in PBS, 1% w/v Triton X-100 for 5 min and then incubated for 1 h at room temperature with the following secondary antibodies: goat anti-rat Alexa Fluor 488 (1:1000, catalog no. A11006, Thermo Fisher Scientific), goat anti-chicken Alexa Fluor 488 (1:1000, catalog no. A11039, Thermo Fisher Scientific), goat anti-mouse Alexa Fluor 594 (1:1000, catalog no. A11032, Thermo Fisher Scientific), goat anti-rabbit Alexa Fluor 594 (1:1000, catalog no. A21429; Thermo Fisher Scientific), goat anti-rat Alexa Fluor 594 (1:1000, catalog no. A21209; Thermo Fisher Scientific). Slides were then washed three times in PBS, 1% w/v Triton X-100 for 5 min and stained with 0.5 μg.mL<sup>−1</sup> DAPI diluted in PBS for 10 min. Slides were washed once then mounted with ProLong Gold Antifade mounting media (catalog no. P36934; Thermo Fisher Scientific). Coverslips were placed on top of the slides and the slides were allowed to cure for 48 hours. Images were captured with an LSM780 confocal microscope (Zeiss) and specimens were scored blindly.</p>
</sec>
<sec id="Sec28">
<title>Immunofluorescence on cultured cells</title>
<p id="Par53">To analyze the localization and stability of C-terminally truncated REV1, the sequences for full length and truncated REV1 were cloned into pEGFP-C1. HEK293 cells were seeded on no. 1.5 coverslips (catalog no. 631-0150, VWR) and 24 h later transfected with either plasmid. Subsequently, cells were washed twice for 5 min with PBS then fixed with 4% paraformaldehyde (catalog no. 43368, Alfa Aesar) for 20 min and washed with PBS twice for 5 min. Cells were then permeabilised for 10 min with PBS containing 0.1% Triton X-100 and washed with PBS twice for 5 min. Cells were then blocked with PBS-S/0.1% Tween 20 (PBS-S-T) containing 5% BSA for 30 min. Cells were then incubated overnight at 4 °C with the following primary antibody diluted in blocking buffer: anti-GFP (1:500, catalog no. GF090Rl; Nacalai). Cells were then washed with PBS and incubated for 1 h at 37 °C with the following secondary antibody diluted in blocking buffer: goat anti-rat Alexa Fluor 594 (1:1000, catalog no. A21209; Thermo Fisher Scientific). After washes with PBS, coverslips were incubated for 10 min with 2 μg/ml DAPI. After washes in PBS, coverslips were mounted on glass slides using Prolong Gold Antifade Mountant (catalog no. P36934, ThermoFisher Scientific). Images were captured with a LSM780 confocal microscope (Zeiss).</p>
</sec>
<sec id="Sec29">
<title>Immunoblotting</title>
<p id="Par54">To demonstrate the stability of C-terminally truncated REV1, the sequences for full length and truncated REV1 were cloned into pExpress with a 2x FLAG tag at the N-termini. HEK293 cells were transfected with either plasmid and were harvested for protein extraction 48 h later. Protein samples were supplemented with LDS buffer (catalog no. NP0007, Themo Fisher Scientific) and 5% β-mercaptoethanol final, boiled for 5 min at 95 °C and resolved by polyacrylamide gel electrophoresis on NuPAGE 4–12%, Bis–Tris, Mini Protein gels (catalog no. NP0321BOX, ThermoFisher Scientific) in MOPS-SDS buffer (50 mM MOPS, 50 mM Tris base, 3.47 mM SDS, 1 mM EDTA). Separated proteins were transferred onto 0.2 μm nitrocellulose membranes (catalog no 10600015, GE Healthcare) in Tris-glycine (25 mM Tris, 192 mM glycine, ph 8.3) buffer with 20% ethanol. Transfer was set at 35 V for 90 min in a Xcell II Blot module (catalog no. EI9051, ThermoFisher Scientific). After transfer, membranes were incubated for 1 h in blocking buffer (Tris-buffered saline, 0.1% Tween 20, 5% non-fat dry milk). Membranes were then incubated with following primary antibodies diluted in blocking buffer and incubated at 4 °C overnight with gentle agitation: anti-FLAG (1:200, M2 clone, catalog no. F1804, Sigma-Aldrich) and anti-β-actin (1:1000, catalog no. ab8227; Abcam). The following secondary antibodies were diluted in blocking buffer and incubated for 1 h at room temperature with gentle agitation: swine anti-rabbit Ig HRP-conjugated (1:3000, catalog no. P0399, Dako) and goat anti-mouse Ig HRP-conjugated (1:5000, catalog no. P0447, Dako).</p>
</sec>
<sec id="Sec30">
<title>Preparation of PGCs and their quantification and isolation by flow cytometry</title>
<p id="Par55">For PGC quantification, the entire embryo (E8.5-10.5) or the developing urogenital ridges (E11.5-13.5) were isolated from embryos and placed into 500 μL or 150 μL pre-warmed trypsin solution (2.5 μg.mL<sup>−1</sup> trypsin (Gibco), 25 mM Tris, 120 mM NaCl, 25 mM KCl, 25 mM KH<sub>2</sub>PO<sub>4</sub>, 25 mM glucose, 25 mM EDTA, pH 7.6) respectively, and incubated at 37 °C for 10 min. Subsequently, 1 μL of Benzonase endonuclease (Millipore) was added and the sample gently disaggregated by pipetting and incubated for 5 min at 37 °C. The trypsin was inactivated by adding 1 mL of PBS/5% v/v fetal bovine serum and centrifuged at 1000 x g for 10 min. The sample was resuspended in 100 μL of anti-SSEA1 conjugated to Alexa Fluor 647 (catalog no. MC-480; Biolegend) diluted 1:100 in PBS/2.5% v/v fetal bovine serum and incubated for 10 min at room temperature. Samples were diluted by adding 300 μL of PBS/2.5% v/v fetal bovine serum and passed through a 70 μm filter. For quantification, 300 μL of the samples were immediately run on an ECLIPSE analyzer (Sony Biotechnology) and the data analyzed using FlowJo v10. For sorting of cells, samples were immediately run on a Synergy cell sorter (Sony Biotechnology), the cells sorted into 10 μL of PBS, centrifuged at 3500 x g for 5 min and stored at −80 °C until further analysis.</p>
</sec>
<sec id="Sec31">
<title>Gene expression analysis</title>
<p id="Par56">The expression of <italic>Rev1</italic>, <italic>Rev7</italic>, <italic>Rev3l, Polk</italic> and <italic>Polq</italic> in PGCs was assessed by droplet digital PCR (ddPCR). PGCs (GOF18-GFP<sup>+</sup>SSEA1<sup>+</sup>) and somatic cells (GOF18-GFP<sup>-</sup>SSEA1<sup>-</sup>) were isolated by FACS as described above. Total RNA was extracted using the PicoPure RNA Isolation Kit (Thermo Fisher Scientific) and first-strand complementary DNA was synthesized using the SuperScript IV Reverse Transcriptase (Thermo Fisher Scientific) according to the manufacturer’s instructions. ddPCR was performed using ddPCR Supermix for Probes (BioRad) on a QX ONE Droplet Digital PCR (BioRad) following the manufacturer’s instructions. Taqman probes were purchased from ThermoFisher; Rev7, Rev3l, Rev1, Polk, Polq labeled with FAM and Gapdh with VIC. The expression was normalized to <italic>Gapdh</italic> and made relative to the somatic cells. The expression of <italic>Rev1</italic> in MEFs was assessed by purifying RNA with the RNeasy kit (Qiagen) according to the manufacturer’s instructions and first-strand complementary DNA was synthesized using the SuperScript IV Reverse Transcriptase (Thermo Fisher Scientific) according to the manufacturer’s instructions. PCR amplification was performed using the TaqMan Fast Advanced Master Mix (Thermo Fisher Scientific). PCR amplification was performed on a ViiA 7 cycler for 95 °C for 15 s and 60 °C for 1 min. Mean threshold cycles were determined from three technical repeats using the comparative CT methodology. All expression levels were normalized to <italic>Gapdh</italic> (mm99999915_g1). To perform gene expression analysis on <italic>Rev1</italic><sup><italic>-/-</italic></sup> and <italic>Pcna</italic><sup><italic>R/R</italic></sup> PGCs, 50 cells were sorted into 10 μL of Single cell lysis buffer supplemented with DNAseI (Single Cell-to-CT Kit ThermoFisher, 4458237) and stored at −80 °C. The lysis and reverse transcription was performed using the single cell-to-CT kit (ThermoFisher, 4458237) according to the manufacturer’s description. Pre-amplification was performed with Taqman Probes (Thermofisher) targeting <italic>Ddx4</italic>, <italic>Nanos3</italic>, <italic>Prdm1</italic> and <italic>Gapdh</italic>. PCR amplification was performed using the TaqMan Fast Advanced Master Mix (Thermo Fisher Scientific). PCR amplification was performed on a ViiA 7 cycler for 95 °C for 15 s and 60 °C for 1 min. Mean threshold cycles were determined from three technical repeats using the comparative C<sub>T</sub> methodology. All expression levels were normalized to <italic>Gapdh</italic>. For expression analysis of <italic>Dnmt1</italic>, <italic>Dnmt3a</italic>, <italic>Dnmt3b</italic>, <italic>Tet1</italic>, <italic>Tet2</italic>, <italic>Stella</italic>, <italic>Fragilis</italic>, <italic>Dazl</italic>, <italic>Mael</italic>, <italic>Mili, Sycp3, Mov10l1, Dazl, Hormad1</italic> and <italic>Brdt</italic> E12.5 wildtype and <italic>Pcna</italic><sup><italic>R/R</italic></sup> PGCs were FACS purified as described above. Subsequently, RNA was prepared using the NEBNext® Single Cell/Low Input RNA Library Prep Kit for Illumina® kit and NEBNext® Poly(a) mRNA Magnetic Isolation module kit following the manufacturer’s instructions. Subsequently, all libraries were ligated to NEBNext® Multiplex Oligos for Illumina® (Dual Index Primer Set I) and quality control was performed using a 2100 Bioanalyser High Sensitivity DNA Kit 5067-4626 (Agilent) and libraries quantified using a Qubit<sup>TM</sup> Fluorometer following the manufacturer’s instructions. Finally, RNA sequencing libraries were pooled to a final concentration of 8.5 nM. RNA expression was performed using Brilliant II SYBR Green QPCR Master Mix (Agilent) using a ViiA 7 Real-Time PCR system (Thermo Fisher Scientific) at 95 °C for 10 min and 40 cycles of 95 °C for 15 s and 60 °C for 1 min. Mean threshold were determined from three technical repeats per sample and oligonucleotide pair using standard comparative C<sub>T</sub> methods. All expression levels were normalized to <italic>Gapdh</italic>. For the testis specific genes <italic>Mov10l1</italic> and <italic>Brdt</italic>, gene expression values were only calculated for male samples.</p>
</sec>
<sec id="Sec32">
<title>Bisulfite sequencing</title>
<p id="Par57">PGCs (GOF18-GFP<sup>+</sup>SSEA1<sup>+</sup>) and somatic cells (GOF18-GFP<sup>-</sup>SSEA1<sup>-</sup>) were isolated from E12.5 embryos carrying the GOF18-GFP reporter by FACS as described above. Genomic DNA extraction and sodium bisulfite conversion was performed using the EZ DNA Methylation-Direct Kit (Zymo Research Cat. No. D5020) following the manufacturer’s instructions. Nested oligonucleotide pairs described previously were used to amplify <italic>Line-1</italic> (<italic>6</italic>), <italic>Dazl</italic> (<italic>68</italic>) and <italic>Mili</italic> (<italic>68</italic>) sequences using the ZymoTaq polymerase (Zymo Research). PCR products were separated out on an agarose gel and gel-purified using the QIAquick Gel Extraction Kit (Qiagen) following the manufacturer’s instructions. Gel extracted products were ligated into the pGEM-T Easy Vector System I (Promega). Ligation products were transformed into <italic>E. coli</italic> and single clones picked following blue-white selection and sent for sequencing using M13R primer. Sequencing reads were analyzed using Quantification for Methylation Analysis (QUMA) software with standard quality control settings.</p>
</sec>
<sec id="Sec33">
<title>Whole genome Bisulfite sequencing</title>
<p id="Par58">PGCs (GOF18-GFP<sup>+</sup>SSEA1<sup>+</sup>) and somatic cells (GOF18-GFP<sup>-</sup>SSEA1<sup>-</sup>) were isolated from E12.5 embryos carrying the GOF18-GFP reporter by FACS as described above. WGBS-sequencing libraries were prepared according to the bulk version of the published scBS-seq protocol<sup><xref ref-type="bibr" rid="CR109">109</xref></sup>. Briefly, cells were lysed in RLT plus buffer (Qiagen) then bisulfite converted using the Zymo EZ-Direct kit. Converted DNA was prepared for Illumina sequencing via two rounds of random primed synthesis using oligos containing Illumina adapter sequences followed by PCR to amplify and incorporate indexes. Sequencing was performed on a NextSeq 500 instrument using 75 bp paired end reads. Bismark v0.23.1 was used to align DNA reads to the bisulfite converted GRCm38 mouse genome then perform methylation calling<sup><xref ref-type="bibr" rid="CR110">110</xref></sup>. E6.5 Epiblast data was taken from our previous work selecting a random subset of 50 E6.5 Epiblast cells<sup><xref ref-type="bibr" rid="CR111">111</xref></sup>. For analysis of genomic features (e.g., promoters, gene bodies, 5Kbp tiles), CpG methylation rates were computed assuming a binomial model as previously<sup><xref ref-type="bibr" rid="CR111">111</xref></sup>.To generate the Circos plot, DNA-methylation bigwig files were read into python using pyBigWig library and resampled to 5 Mbp bins. The chromosome band data is obtained from the UCSC table “cytoBandIdeo” of the GRCm38/mm10 assembly. The Circos plot was generated using the pycircos library.</p>
</sec>
<sec id="Sec34">
<title>EdU incorporation analysis in vivo</title>
<p id="Par59">To assess the in vivo incorporation of 5-ethynyl-2´-deoxyuridine (EdU) into the DNA of PGCs (GOF18-GFP<sup>+</sup>) the Click-iT™ Plus EdU Cell Proliferation Kit for Imaging, Alexa Fluor™ 594 kit (Catalog number: C10639) was used. Pregnant mice were given a single dose of EdU (50 mg.kg<sup>−1</sup>) by intraperitoneal injection (IP) at 10 ml.kg<sup>−1</sup>. Females were subsequently culled 4 h post-IP at E12.5 and the embryos harvested; the fetal gonads were dissected and placed into ice-cold PBS. Fetal gonads were fixed in PBS, 4% w/v paraformaldehyde for 30 min at 4 °C. Fixed samples were washed once in PBS for 5 min and then three times in PBS, 1% w/v Triton X-100 for 15 min at room temperature. Samples were then pressed onto glass slides and a large hydrophobic ring drawn around the sample before being incubated in blocking buffer (PBS, 1% w/v Triton X-100, 1% w/v BSA) for 1 h at room temperature. Samples were then incubated with anti-GFP (1:500, catalog no. GF090R; Nacalai) diluted in blocking buffer overnight at 4 °C. Subsequently, samples were washed three times in PBS, 1% w/v Triton X-100 for 5 min at room temperature and fixed in PBS, 2% w/v paraformaldehyde for 20 min at room temperature. Slides were washed three times in PBS and incubated in 200 μL of Click-iT® Plus reaction cocktail made per manufacturer’s instructions. Samples were washed three times in PBS, 1% w/v Triton X-100 and incubated with goat anti-rat Alexa Fluor 488 (1:1000, catalog no. A11029, Thermo Fisher Scientific) secondary antibody. Slides were washed three times in PBS, 1% w/v Triton X-100 for 5 min and stained with DAPI (0.5 μg.mL<sup>−1</sup>, PBS) before being mounted with ProLong Gold Antifade Mountant (catalog no. P36934; Thermo Fisher Scientific). Coverslips were placed on top of the slides and allowed to cure for 48 h before images were captured with an LSM780 confocal microscope (Zeiss) and the frequency of positive cells scored blindly.</p>
</sec>
<sec id="Sec35">
<title>Blood counts</title>
<p id="Par60">A 50 μL total blood sample was taken from saphenous veins or via cardiac puncture and transferred into a K3EDTA MiniCollect tubes (Greiner bio-one) and analysed on a VetABC analyzer, using standard settings for mice (Horiba).</p>
</sec>
<sec id="Sec36">
<title>Serum hormone analysis</title>
<p id="Par61">For the determination of serum luteinizing hormone, follicle stimulating hormone (FSH) and testosterone levels, blood samples were collected as described above and transferred into a Microvette collection tube (SARSTEDT) and centrifuged at 10,000 x g. for 10 min at room temperature. The supernatant (serum) was transferred to a 1.5 mL eppendorf tube and stored at −80 °C until further analysis. For serum LH and FSH concentrations were determined using the Milliplex Map Mouse Pituitary Magnetic Bead Panel (catalog no. MPTMAG-49K). Plates were loaded manually and washed using a Bio-Plex Pro wash station (BioRad) and data analysis performed using a Magpix Multiplexer reader (BioRad). Serum testosterone levels were determined using the Demeditec Diagnostics rat/mouse ELISA kit (catalog no. DEV9911). Samples were loaded manually and washes performed using a WellWash Versa platewasher (Thermo Scientific). Absorbance was measured at 450 nm using Perkin Elmer Multicalc software. Serum levels of urea, creatinine, aspartate aminotransferase and albumin were measured using a Siemens Dimension RxL analyser.</p>
</sec>
<sec id="Sec37">
<title>Haematopoiesis analysis</title>
<p id="Par62">Flow cytometry was performed on bone marrow cells that were isolated from the femora and tibiae of mutant mice and appropriate controls by flushing cells and passing them through a 70-μm filter. The following antibodies were used to stain for HSCs: FITC-conjugated lineage cocktail with antibodies anti-CD4 (clone H129.19, BD Pharmingen), CD3e (clone 145-2C11, eBioscience), Ly-6G/Gr-1 (clone RB6-8C5, eBioscience), CD11b/Mac-1 (clone M1/70, BD Pharmingen), CD45R/B220 (clone RA3-6B2, BD Pharmingen), Fcε R1α (clone MAR-1, eBioscience), CD8a (clone 53-6.7, BD Pharmingen), CD11c (clone N418, eBioscience) and TER-119 (clone Ter119, BD Pharmingen), anti-c-Kit (PerCP-Cy5.5, clone 2B8, eBioscience), anti-Sca-1 (PE-Cy7, clone D7, eBioscience). When staining for SLAM markers the same lineage cocktail was used (FITC) with the addition of the following antibodies: anti-CD48 (FITC, clone HM48-1, BioLegend), anti-CD41 (FITC, clone MWReg30, BD Pharmigen), anti-CD150 (APC, clone TC15-12F12.2, BioLegend) and anti-c-Kit and Sca-1 as above. Maturation of B cells was assessed using anti-CD45R/B220 (PE, clone RA3-6B2, BD Pharmingen) and anti-IgM (APC, clone II/41, BD Pharmingen). The maturation of the erythroid lineage was analysed using antibodies anti-TER-119 (APC, clone Ter-119, BD Pharmingen) and anti-CD71 (PE, clone C2, BD Pharmingen). Granulocyte–macrophage maturation was assessed with antibodies anti-CD11b/Mac-1 (APC, clone M1/70, BD Pharmingen) and anti-Ly-6G/Gr-1 (PE, clone RB6-8C5, eBioscience). Thymic T-cell maturation was assessed using CD4 (FITC, clone H129.19, BD Pharmingen) and CD8a (PE, clone 53-6.7, BD Pharmingen) antibodies. The samples were incubated for 15 min at 4 °C in the dark with the exception of samples containing anti-CD34 (RAM34), which were incubated for 90 min. Samples were run on a LSRII flow cytometer (BD Pharmingen) and the data were analysed with FlowJo v10.</p>
</sec>
<sec id="Sec38">
<title>Micronucleus assay</title>
<p id="Par63">The micronucleus assay<sup><xref ref-type="bibr" rid="CR112">112</xref>,<xref ref-type="bibr" rid="CR113">113</xref></sup> was performed by bleeding mice; 62 μl blood was mixed with 338 μl PBS supplemented with 1000 U ml<sup>−1</sup> of heparin (Calbiochem). 360 μl of blood suspension was then added to 3.6 ml of methanol at −80 °C and stored at −80 °C for at least 12 h. 1 ml of fixed blood cells was then washed with 6 ml of bicarbonate buffer (0.9% NaCl, 5.3 mM NaHCO3). The cells were resuspended in 150 μl of bicarbonate buffer and 20 μl of this suspension was used for subsequent staining. 72 μl of bicarbonate buffer, 1 μl of FITC-conjugated CD71 antibody (GenTex, clone R17217.1.4) and 7 μl RNase A (Sigma) were premixed and added to 20 μl of each cell suspension. The cells were stained at 4 °C for 45 min, followed by addition of 1 ml bicarbonate buffer and centrifugation. Finally, cell pellets were resuspended in 500 μl bicarbonate buffer supplemented with 5 μg ml<sup>−1</sup> propidium iodide (Sigma). The samples were analysed immediately on an LSRII FACS analyser (BD) and the data analysed with FlowJo v10.</p>
</sec>
<sec id="Sec39">
<title>Statistics and reproducibility</title>
<p id="Par64">The number of independent biological samples and technical repeats (<italic>n</italic>) are indicated in figure legends. Unless otherwise stated, data are shown as mean ± standard deviation (s.d.) and the nonparametric two-tailed Mann–Whitney test was employed to determine statistical significance. Analysis was performed in GraphPad Prism version 8. The images in Figs. <xref ref-type="fig" rid="Fig1">1l</xref> and <xref ref-type="fig" rid="Fig3">3i</xref> are representative of the following number of observations (Fig. <xref ref-type="fig" rid="Fig1">1l</xref>: Wildtype –7, <italic>Rev1</italic><sup><italic>-/-</italic></sup> - 12; Fig. <xref ref-type="fig" rid="Fig3">3i</xref>: Wildtype – 18, <italic>Pcna</italic><sup><italic>R/R</italic></sup> – 21).</p>
</sec>
<sec id="Sec40">
<title>Reporting summary</title>
<p id="Par65">Further information on research design is available in the <xref ref-type="media" rid="MOESM3">Nature Portfolio Reporting Summary</xref> linked to this article.</p>
</sec>
</sec>
<sec sec-type="supplementary-material">
<sec id="Sec41">
<title>Supplementary information</title>
<p>
<supplementary-material content-type="local-data" id="MOESM1"><media xlink:href="41467_2024_47219_MOESM1_ESM.pdf"><caption><p>Supplementary Information</p></caption></media></supplementary-material>
<supplementary-material content-type="local-data" id="MOESM2"><media xlink:href="41467_2024_47219_MOESM2_ESM.pdf"><caption><p>Peer Review File</p></caption></media></supplementary-material>
<supplementary-material content-type="local-data" id="MOESM3"><media xlink:href="41467_2024_47219_MOESM3_ESM.pdf"><caption><p>Reporting Summary</p></caption></media></supplementary-material>
</p>
</sec>
<sec id="Sec42">
<title>Source data</title>
<p>
<supplementary-material content-type="local-data" id="MOESM4"><media xlink:href="41467_2024_47219_MOESM4_ESM.zip"><caption><p>Source Data</p></caption></media></supplementary-material>
</p>
</sec>
</sec>
</body>