<body>
<sec id="Sec1">
<title>Background</title>
<p id="Par5">The issue of antibiotic resistance poses a significant challenge to world health. Antimicrobial resistance (AMR) refers to the emergence of antibiotic resistance in microorganisms [<xref ref-type="bibr" rid="CR1">1</xref>]. The emergence of antibiotic resistance might occur as a result of the continuous abuse of antibiotics [<xref ref-type="bibr" rid="CR2">2</xref>]. The level of opposition is increasing significantly across many regions. The proliferation of new resistance mechanisms worldwide has led to a diminishing success rate in treating common viral illnesses. As a result, managing diseases resulting from bacterial infections has become more intricate and sometimes unattainable [<xref ref-type="bibr" rid="CR3">3</xref>]. The Centers for Disease Control and Prevention (CDC) reports that illnesses caused by bacteria that are resistant to antibiotics lead to an annual incidence of 2.8 million cases and a mortality rate exceeding 35,000 [<xref ref-type="bibr" rid="CR4">4</xref>].</p>
<p id="Par6">The isolated clinical strains of Klebsiella pneumoniae are a notable contributor to nosocomial infections due to their extensive antibiotic resistance [<xref ref-type="bibr" rid="CR5">5</xref>]. Moreover, Klebsiella pneumoniae is accountable for infections obtained within the community, particularly among those who have weakened immune systems or pre-existing medical disorders. K. pneumoniae infections have limited treatment options due to the development of resistance to β-lactam antibiotics [<xref ref-type="bibr" rid="CR6">6</xref>]. In addition, Klebsiella pneumoniae, a natural producer of β lactamases, particularly extended-spectrum β-lactamases (ESBLs), has been shown to contribute to the development of antibiotic resistance [<xref ref-type="bibr" rid="CR7">7</xref>]. Various approaches, such as botanical extracts and pharmaceuticals, have been used to address Klebsiella pneumoniae infection, yet the majority of these strategies have been ineffective. Klebsiella pneumoniae exhibits resistance to beta-lactam antibiotics. Consequently, this infection needs innovative treatment strategies [<xref ref-type="bibr" rid="CR7">7</xref>].</p>
<p id="Par7">Vaccination is an effective method to combat K. pneumoniae infections, contributing to antibiotic resistance development. Despite current studies on K. pneumoniae vaccines, none have received official approval [<xref ref-type="bibr" rid="CR8">8</xref>]. Hence, in order to address this matter of public health, expeditious intervention is imperative. Despite their limited immunogenicity, pure polysaccharide vaccines have shown notable efficacy in providing 80% protection against isolates of K. pneumoniae, as evidenced by the findings of a study on a 24-valent capsular polysaccharide (SPC) vaccine [<xref ref-type="bibr" rid="CR9">9</xref>]. The efficacy of vaccinations in mitigating antibiotic-resistant bacterial infections, such as K. pneumoniae, has been well acknowledged. However, their significance in managing these diseases has traditionally been underestimated [<xref ref-type="bibr" rid="CR10">10</xref>].</p>
<p id="Par8">Consequently, the development of vaccinations via technological progress will provide a viable resolution to the issue of antibiotic resistance. The development of vaccines has seen significant advancements as a result of progress in computer technology and genetic data. The field of reverse vaccinology (RV) employs a genome-based methodology to facilitate the creation of novel vaccine candidates. Recent studies have shown the beneficial use of RV, highlighting the need for vaccination as a preventive and cost-effective strategy against this virus. Researchers used reverse vaccinology and subtractive proteomics techniques in this study to ascertain prospective therapeutic interventions for A. baumannii [<xref ref-type="bibr" rid="CR11">11</xref>, <xref ref-type="bibr" rid="CR16">16</xref>, <xref ref-type="bibr" rid="CR43">43</xref>]. The recombinant chimeric peptide vaccine demonstrated a 50% efficacy in hamsters against the pathogen during recent in vitro testing [<xref ref-type="bibr" rid="CR12">12</xref>]. In addition, pan-genome-based RV (PGRV) development has been undertaken in response to the genomic variety seen in bacterial pathogens [<xref ref-type="bibr" rid="CR13">13</xref>]. This approach aims to identify significant proteome antigens.</p>
<p id="Par9">The efficacy of a subunit vaccine, which comprises an adjuvant, linker, and epitope, has been shown. Bioinformatics is of utmost importance in developing vaccines since it involves identifying and predicting key components, including the target antigen, B and T epitopes, and a suitable linker. Protein antigenicity plays a crucial role in shaping the design of vaccines. The accurate prediction of the subcellular localization of proteins is of utmost importance due to the immune system’s ability to promptly detect proteins present on the surface of pathogens. Klebsiella’s most notable surface proteins are MrkD, Iron-regulated outer membrane proteins, RmpA, YaiO, EMG21, TonB, ompA, ydiY, and OMVs. The immunogenicity of these proteins is attributed to their robust interaction with the immune system, which is facilitated by their positioning on the outer membrane [<xref ref-type="bibr" rid="CR14">14</xref>–<xref ref-type="bibr" rid="CR17">17</xref>].</p>
<p id="Par10">Other approaches, like pang-genomics, immunoinformatic, and biophysical studies, may be used with the RV methodology to create a distinct multi-epitope polypeptide [<xref ref-type="bibr" rid="CR14">14</xref>, <xref ref-type="bibr" rid="CR15">15</xref>]. After prioritizing different epitopes, a pan-proteome screening approach was used to ascertain the essential vaccination proteins [<xref ref-type="bibr" rid="CR16">16</xref>]. Furthermore, the host immune system was guaranteed to accurately identify and handle the chosen epitopes, which are crucial for the pathogen’s survival and do not have any similar structures in humans. In order to ascertain immune responses specific to pathogens, conserved epitopes that have been shown to induce distinct B-cell responses were chosen.</p>
</sec>
<sec id="Sec2">
<title>Methods</title>
<sec id="Sec3">
<title>Retrieval and analysis of protein sequences</title>
<p id="Par12">The National Center for Biotechnology Information (NCBI) website (<ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/">https://www.ncbi.nlm.nih.gov/</ext-link>) provided the <italic>Klebsiella pneumoniae</italic>. The selected strain’s genus, genome, and proteome were further investigated utilizing the web proteomics program Vaxign (<ext-link ext-link-type="uri" xlink:href="https://violinet.org/vaxign/">https://violinet.org/vaxign/</ext-link>). The UniProt collection (<ext-link ext-link-type="uri" xlink:href="https://www.uniprot.org/">https://www.uniprot.org/</ext-link>) was used to retrieve the complete proteome of this particular bacteria. A FASTA file extract was created using the selected protein sequences. Next, the VaxiJen v2.0 webpage (<ext-link ext-link-type="uri" xlink:href="http://www.ddgpharmfac.net/vaxijen/">http://www.ddgpharmfac.net/vaxijen/</ext-link>) was used to evaluate the immunogenicity of the antigens utilizing the standard threshold value. This server’s auto cross-covariance (ACC) conversion technique yields a 70–89% accurate prediction result. After that, the AllergenFP v1.0 service (<ext-link ext-link-type="uri" xlink:href="https://ddgpharmfac.net/AllergenFP/">https://ddgpharmfac.net/AllergenFP/</ext-link>) was used to determine the allergenic patterns of the selected antigens. With an accuracy of 88.9%, this server employs a novel descriptive adjective fingerprint method that is alignment-free. For determining the transmembrane (TM) helix, the TMHMM v2.0 platform—which depends on a hidden Markov model (HMM)—was used. Protein molecules with fewer TM helices have been selected for the following investigation stage because they are immunological and non-allergic.</p>
</sec>
<sec id="Sec4">
<title>Forecasting the qualities of virulence, antigenicity, flexibility, and hydrophilicity</title>
<p id="Par13">The pathogenicity characteristic of potential vaccine candidates was examined using the VirulentPred database (<ext-link ext-link-type="uri" xlink:href="http://203.92.44.117/virulent/">http://203.92.44.117/virulent/</ext-link>). The cascade SVM module with a threshold value of 0.5 (≥ 0.5) was used in this investigation. The database VaxiJen provides a reliable way to determine the antigenic characteristics of proteins. This server was used to describe the antigenic characteristics of proteins solely based on their physicochemical properties, as opposed to sequence alignment. The antigenicity of the chosen proteins was determined using the VaxiJen method (<ext-link ext-link-type="uri" xlink:href="http://www.ddg-pharmfac.net/vaxijen/VaxiJen/VaxiJen.html">http://www.ddg-pharmfac.net/vaxijen/VaxiJen/VaxiJen.html</ext-link>), with a cut-off of 0.5. The median flexibility and hydrophilicity of proteins were also calculated using <ext-link ext-link-type="uri" xlink:href="http://tools.immuneepitope.org">http://tools.immuneepitope.org</ext-link>. A protein’s degree of solubility may provide helpful information about how effectively it performs. Over thirty per cent of the molecules created in immunization efforts are insoluble. The SOLpro (<ext-link ext-link-type="uri" xlink:href="https://protein-sol.manchester.ac.uk/">https://protein-sol.manchester.ac.uk/</ext-link>) tool uses a two-step SVM technique to estimate protein solubility. As a result, SOLpro calculates the proteins’ solubility.</p>
</sec>
<sec id="Sec5">
<title>Cytotoxic T lymphocyte (CTL) epitope identification and analysis</title>
<p id="Par14">The cytotoxic T lymphocytes (CTLs) are critical components of the host defence system. Direct contact between these cells and infectious germs inside the body’s immune response may lead to their death. The NetCTL v1.2 webpage (<ext-link ext-link-type="uri" xlink:href="http://www.cbs.dtu.dk/services/NetCTL/">http://www.cbs.dtu.dk/services/NetCTL/</ext-link>) was utilized to run the selected protein sequences to anticipate the CTL epitopes with high combination scores. To maintain the sensitivity and specificity levels at 0.74 and 0.98, respectively, a threshold level of 0.90 was used. The MHC-I binding program available on the IEDB webpage (<ext-link ext-link-type="uri" xlink:href="http://tools.iedb.org/mhci/">http://tools.iedb.org/mhci/</ext-link>) was used to identify the MHC-I binding variants of each CTL epitope following the CONSENSUS approach. It was believed that CTL epitopes might be lowered if their percentile rank was equal to or less than 2 because a lower percentile rank indicates a stronger affinity. Subsequently, the VaxiJen v2.0 web page (<ext-link ext-link-type="uri" xlink:href="http://www.ddg-pharmfac.net/vaxijen/">http://www.ddg-pharmfac.net/vaxijen/</ext-link>), the AllerTOP v2.0 programs (<ext-link ext-link-type="uri" xlink:href="https://www.ddg-pharmfac.net/AllerTOP/">https://www.ddg-pharmfac.net/AllerTOP/</ext-link>), the ToxinPred applications (<ext-link ext-link-type="uri" xlink:href="http://crdd.osdd.net/raghava/toxinpred/">http://crdd.osdd.net/raghava/toxinpred/</ext-link>), and the IEDB Class I Immunogenicity system (<ext-link ext-link-type="uri" xlink:href="http://tools.iedb.org/immunogenicity/">http://tools.iedb.org/immunogenicity/</ext-link>) were used to determine antigenicity, the allergenic arrangement, toxic effects assessment, and immunogenicity for each CTL epitope, respectively.</p>
<p id="Par15">Using fivefold cross-validation, the AllerTop v2.0 program uses k-nearest neighbor (kNN) methods, amino acid identities, and ACC modification techniques to identify non-allergens within allergens with an 85.3% predicted performance. The ToxinPred service evaluates the properties of different proteins using support vector machines (SVM), a computational method, and a statistical toxicity prediction model. Immunogenicity modelling aimed to ascertain if an epitope would elicit an immune response. While developing vaccines, CTL immunostimulatory, non-toxic, non-allergenic, and antigenic epitopes with high C-scores are considered.</p>
<sec id="Sec6">
<title>Helper T-Lymphocyte (HTL) epitope identification and evaluation</title>
<p id="Par16">Helper T lymphocytes (HTLs) are a component of the immune system’s adaptive response that may recognize foreign antigens and activate B-cells and CTLs to destroy harmful microbes. Among the selected sequence data, 15-mer HTL epitopes were found using the MHC-II binding program of the IEDB collection (<ext-link ext-link-type="uri" xlink:href="http://tools.iedb.org/mhcii/">http://tools.iedb.org/mhcii/</ext-link>). We used the CONSENSUS method to determine the corresponding interaction alleles of the compounds while keeping consistency in mind, with a percentile rank cutoff of equal to or less than 2. For every HTL epitope, allergenicity, antigenicity, and toxicity were predicted using the AllerTOP v2.0 web page, VaxiJen v2.0 program, and ToxinPred webpage.</p>
<p id="Par17">The ability to trigger cytokines was used to accurately evaluate antigenic, non-toxic, and allergic HTL epitopes. IFN-γ activates natural killer cells and macrophages, which may trigger specialized and native immune reactions. It also enhances MHC responsiveness to ligands and plays a critical role in inhibiting the growth of microorganisms. We utilized the 81.39% efficient IFN-epitope website (<ext-link ext-link-type="uri" xlink:href="http://crdd.osdd.net/raghava/ifnepitope/">http://crdd.osdd.net/raghava/ifnepitope/</ext-link>) to estimate interferon-gamma (IFN-γ). Furthermore, the websites IL4pred (<ext-link ext-link-type="uri" xlink:href="https://webs.iiitd.edu.in/raghava/il4pred/">https://webs.iiitd.edu.in/raghava/il4pred/</ext-link>) and IL10pred (<ext-link ext-link-type="uri" xlink:href="https://webs.iiitd.edu.in/raghava/il10pred/">https://webs.iiitd.edu.in/raghava/il10pred/</ext-link>) were used to evaluate the inducing qualities of interleukin-4 (IL-4) and IL-10 (IL-10), respectively. For both servers, the threshold values were − 0.3. Both procedures were executed using SVM-based methods. The IL4pred and IL10pred technologies give 75.76 and 81.24% accuracy, respectively. We considered the generational capability of all three cytokines while selecting the HTL epitopes for the vaccination. We prioritized IFN induction and the IL4- or IL10-inducing qualities of the HTL epitopes for antigens where no epitope exhibited all three cytokine-inducing capacities.</p>
</sec>
</sec>
<sec id="Sec7">
<title>Linear B-lymphocyte (LBL) epitope identification and evaluation</title>
<p id="Par18">B-cell epitopes are the primary agents that cause humoral or particular antibody responses. Using the ABCpred website (<ext-link ext-link-type="uri" xlink:href="https://webs.iiitd.edu.in/raghava/abcpred/">https://webs.iiitd.edu.in/raghava/abcpred/</ext-link>), the LBL epitopes among the selected sequence information were predicted. The web servers VaxiJen v2.0 (<ext-link ext-link-type="uri" xlink:href="http://www.ddg-pharmfac.net/vaxijen/">http://www.ddg-pharmfac.net/vaxijen/</ext-link>), AllerTOP v2.0 (<ext-link ext-link-type="uri" xlink:href="https://www.ddg-pharmfac.net/AllerTOP/">https://www.ddg-pharmfac.net/AllerTOP/</ext-link>), and ToxinPred v2.0 (<ext-link ext-link-type="uri" xlink:href="http://crdd.osdd.net/raghava/">http://crdd.osdd.net/raghava/</ext-link>) were used to evaluate the antigenic, allergic, and toxic patterns of the predicted LBL epitopes. In addition, the iBCE-EL platform’s default parameters (<ext-link ext-link-type="uri" xlink:href="http://www.thegleelab.org/iBCE-EL/">http://www.thegleelab.org/iBCE-EL/</ext-link>) were used to estimate the probability scores of LBLs.</p>
</sec>
<sec id="Sec8">
<title>Epitope conservation and determining human similarity</title>
<p id="Par19">The shortlisted MHC-I and MHC-II epitopes were examined using the IEDB tools. The similarity of the epitope to the human proteome was evaluated to identify which epitope is homologous to the protein database. Since homologous epitopes would not be able to elicit an immune system response, only non-homologous epitopes were used. The human homology was ascertained using the protein BLAST module of the BLAST database (<ext-link ext-link-type="uri" xlink:href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">https://blast.ncbi.nlm.nih.gov/Blast.cgi</ext-link>). This research compared Homo sapiens (taxid: 9606) against other species using default values, with a threshold e-value of 0.05. Epitopes were categorized as non-homologous peptides when no hits were found below the threshold e-value.</p>
</sec>
<sec id="Sec9">
<title>Docking investigations and peptide modelling</title>
<p id="Par20">CTL epitopes were docked with the HLA interacting allele to evaluate the identified CTL epitopes’ ability to bind. The PEP-FOLD v3.5 webpages (<ext-link ext-link-type="uri" xlink:href="https://bioserv.rpbs.univ-paris-diderot.fr/services/PEPFOLD3/">https://bioserv.rpbs.univ-paris-diderot.fr/services/PEPFOLD3/</ext-link>) include 200 simulations applied using the sOPEP sorting approach for the respective CTL epitopes. The sOPEP sorting service uses a de novo approach to determine the structural changes of small peptides (5–50 amino acids). Conformational changes are predicted using the Taboo/Backtrack sampling approach. This service was used to determine the probable structure of each peptide sequence. The efficiency of each structure was assessed using SWISS-PDB VIEWER, and the complex with the lowest energy was selected for further analysis. The co-crystallized ligand epitopes were selected as a control sample for the docking techniques, and human HLA-A*03:01 (PDB ID: 6O9B), HLA-A*01:01 (PDB ID: 6AT9), HLA-B*35:01 (PDB ID: 1A1N), and HLA-B*14:02 (PDB ID: 3BXN) variations were selected as CTL epitopes. The co-crystallized receptor version’s HLA allele composition was obtained from the RCSB Protein Data Library (<ext-link ext-link-type="uri" xlink:href="https://www.rcsb.org/">https://www.rcsb.org/</ext-link>). In order to create proteins, ligands had to be removed from the molecules and hydrogens and Gasteiger-Marsili charges had to be added. UCSF Chimera’s protein synthesis wizard (version 1.11.2) was used to insert hydrogen. Next, Open-Babel was used to convert this file into the pdbqt format. The ligand energy form was optimized and transferred to the pdbqt version using PyRx 0.8’s OpenBabel package. The docking parameters were left at standard in the present operation. The grid box dimensions for HLA-A*01:01, HLAB* 35:01, HLA-A*03:01, and HLA-B*14:02 were set to 60.64 × 73.76 × 45.49 Å, 62.33 × 45.41 × 70.54 Å, 73.20 × 52.65 × 53.24 Å, and 61.25 × 48.69 × 72.95 Å for the X, Y, and Z axes of AutoDock Vina. Lower docking scores indicate a stronger binding affinity. The molecular docking results are expressed as negative quantities in kcal mol − 1. The graphics were created using Microsoft PowerPoint 2019 and UCSF Chimera, and the molecular dynamics results were shown using Discovery Studio (DS) version 4.5.</p>
</sec>
<sec id="Sec10">
<title>MHC clustering algorithm</title>
<p id="Par21">The MHC genomic region shows widespread variation in most mammals. The human MHC genetic code (HLA) is highly variable, with hundreds of alleles. Most of the time, how specifically the MHC allele may work still needs to be determined. MHC allele clustering makes it possible to distinguish between the two types of MHC molecules by comparing their binding-specific traits. Using phylogenetic trees and default settings, heatmaps showing the functional clustering between MHC variants were produced using the MHC-cluster 2.0 webpage (<ext-link ext-link-type="uri" xlink:href="http://www.cbs.dtu.dk/services/MHCcluster/">http://www.cbs.dtu.dk/services/MHCcluster/</ext-link>). The HLA-prevalent and -characterized component was used to construct the NetMHCpan-2.8 approach for MHC class I cluster assessment. In contrast, the pertinent DRB allele elements were used for MHC class II clustering techniques.</p>
</sec>
<sec id="Sec11">
<title>Construction of the vaccine concept</title>
<p id="Par22">A vaccine was developed using the selected proteins’ particular LBL, CTL, and HTL epitopes. We connected the selected HTL, CTL, and LBL using HEYGAEALERAG and Gly-Gly-Gly-Ser (GGGS) linkers. The GGGS linker improves immunogenicity and epitope presentation, affecting binding affinity and protein stability. The HEYGAEALERAG linker promotes immunological processing and inhibits the formation of “junctional epitopes.” The HEYGAEALERAG linker was inserted to act as an interface between CTL and HTL epitopes.</p>
</sec>
<sec id="Sec12">
<title>Assessment of the physicochemical, antigenicity, allergenicity, and solubility profiles</title>
<p id="Par23">Using the ProtParam website (<ext-link ext-link-type="uri" xlink:href="https://web.expasy.org/protparam/">https://web.expasy.org/protparam/</ext-link>), the physicochemical properties of the vaccine construct were examined. These properties included the molecular weight (MW), conceptual isoelectric point (pI), aliphatic index (AI), instability index (II), half-life (both In vitro and In vivo), and grand average hydropathicity (GRAVY). Various techniques were used to investigate the vaccine’s immunologic, allergic, and hydrophilic properties. The ANTIGENPro application from the Scratch protein prediction site and the Vaxijen v2.0 website (accessible at <ext-link ext-link-type="uri" xlink:href="http://www.ddg-pharmfac.net/vaxijen/">http://www.ddg-pharmfac.net/vaxijen/</ext-link>) were also used to predict antigenicity. Cross-validation studies on the pooled dataset showed that ANTIGENPro was 76% accurate. AllerTop v2.0, AllergenFP v1.0, and AlgPred websites (<ext-link ext-link-type="uri" xlink:href="http://crdd.osdd.net/raghava/algpred/">http://crdd.osdd.net/raghava/algpred/</ext-link>) were used to simulate the allergenic characteristics of the vaccine design in light of the expectation of no allergic reaction. The solubility of the vaccine design was assessed using the Scratch protein prediction service’s SOLpro online software (<ext-link ext-link-type="uri" xlink:href="http://scratch.proteomics.ics.uci.edu/">http://scratch.proteomics.ics.uci.edu/</ext-link>). The vaccination is presumed to be soluble if the prediction value is 0.5 or above. Utilizing the Protein-Sol website (<ext-link ext-link-type="uri" xlink:href="https://protein-sol.manchester.ac.uk/">https://protein-sol.manchester.ac.uk/</ext-link>) also improved my understanding of solubility. The protein is soluble if the projected value is more than 0.45. The number of transmembrane helices was predicted using the TMHMM v2.0 webpage (<ext-link ext-link-type="uri" xlink:href="http://www.cbs.dtu.dk/services/TMHMM/">http://www.cbs.dtu.dk/services/TMHMM/</ext-link>). Verify 3D and ERRAT were two validation tools used to assess the tertiary structure’s quality. RMSD was employed as a quantitative metric to assess the similarity between protein structures, density, temperature, and total energy plots.</p>
</sec>
<sec id="Sec13">
<title>In-silico cloning and codon optimization of a proposed vaccine</title>
<p id="Par24">Wrangler (<ext-link ext-link-type="uri" xlink:href="https://www.mrc-lmb.cam.ac.uk/ms/methods/codon.html">https://www.mrc-lmb.cam.ac.uk/ms/methods/codon.html</ext-link>) and JCAT (<ext-link ext-link-type="uri" xlink:href="http://www.jcat.de/">http://www.jcat.de/</ext-link>) web server applications were used to optimize heterologous protein production in the mouse model. The RF-Cloning technology (<ext-link ext-link-type="uri" xlink:href="https://rf-cloning.org/">https://rf-cloning.org/</ext-link>) was used to guarantee vaccine expression, and the final vaccine constructs DNA sequence was created in vector pcDNA3.1(+). Numerous enzyme restriction sites, autonomous transcription terminators, and prokaryotic ribosome receptors were also created.</p>
</sec>
</sec>
<sec id="Sec14" sec-type="results">
<title>Results</title>
<sec id="Sec15">
<title>A potential vaccine candidate has been recognized</title>
<p id="Par25">For this investigation, 68 completely sequenced <italic>Klebsiella pneumoniae</italic> genomes were obtained from the NCBI. Complete proteome information for the strain may be found at <ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/genome/?term=Klebsiella+pneumoniae">https://www.ncbi.nlm.nih.gov/genome/?term=Klebsiella+pneumoniae</ext-link>. The genomes of <italic>Klebsiella pneumoniae</italic> utilized are available at the following link: <ext-link ext-link-type="uri" xlink:href="https://www.ncbi.nlm.nih.gov/genome/browse#!/prokaryotes/815/">https://www.ncbi.nlm.nih.gov/genome/browse#!/prokaryotes/815/</ext-link>. Every genome encoded around 5.59628 megabytes of proteins. Each strain had a different quantity of proteins, but the average protein number was 5301. The number of genomes identified in each strain of <italic>Klebsiella pneumoniae</italic> is shown in Fig. <xref ref-type="fig" rid="Fig1">1</xref>. The core-pan plot indicates that, due to chromosomal flexibility, the pathogen’s expected proteome is open and that the probability of acquiring new genes over time is relatively high.</p>
<p id="Par27">
<fig id="Fig1"><label>Fig. 1</label><caption><p>Genome assembly and annotation report from 86 <italic>Klebsiella pneumoniae</italic> strains</p></caption><graphic id="d33e558" xlink:href="12865_2024_617_Fig1_HTML"></graphic></fig>
</p>
<p id="Par26">Moreover, examining COG patterns demonstrated that the core proteins mostly mediated the metabolic biogenesis. Data processing and storage were associated with the distinct set of proteins present in <italic>Klebsiella pneumoniae</italic> subsp. pneumoniae HS11286 (reference strain). Four categories may be used to categorize the data: recombination, transcription and translation, replication, and RNA processing (Table <xref ref-type="media" rid="MOESM1">S1</xref>).</p>
<p id="Par28">A dataset from Vaxign identified 55 proteins with remarkable characteristics. Twenty-three proteins with possible pathogenicity were then selected. Following an assessment of the antigenicity of proteins, eight possible vaccine candidates remained. Table <xref ref-type="table" rid="Tab1">1</xref> shows this technique in action. The accession number of the proteins was selected on the NCBI webpage after the Uniport databases confirmed the presence of eight different proteins in <italic>K. pneumoniae</italic>. They were classified as extracellular, outer membrane, periplasmic, inner membrane, and cytoplasmic protein types based on the results of the CELLO algorithm (Fig. <xref ref-type="fig" rid="Fig2">2</xref>). The specificity of the proteins chosen for <italic>K. pneumoniae</italic> was confirmed by BLASTp findings, reducing the likelihood that the immunization may interact with host cells. Table <xref ref-type="table" rid="Tab1">1</xref> displays the results of the first screening of eight <italic>K. pneumoniae</italic> proteins. Based on the preliminary screening findings, MrkD, Iron-regulated outer membrane proteins, and RmpA proteins were the most appropriate antigens chosen for immunogenicity (Table <xref ref-type="table" rid="Tab1">1</xref>).</p>
<p id="Par11">
<fig id="Fig2"><label>Fig. 2</label><caption><p>The reverse vaccinology technique was utilized to identify new prospective <italic>Klebsiella pneumoniae</italic> vaccine candidates</p></caption><graphic id="d33e604" xlink:href="12865_2024_617_Fig2_HTML"></graphic></fig>
</p>
<p id="Par29">
<table-wrap id="Tab1"><label>Table 1</label><caption><p>Screening of <italic>Klebsiella pneumoniae</italic> proteins</p></caption><table frame="hsides" rules="groups"><thead><tr><th align="left" rowspan="2">Name</th><th align="left" rowspan="2">Accession Number</th><th align="left" colspan="2">CELLO analysis</th><th align="left" colspan="2">Protein- Sol</th><th align="left">IEDB</th><th align="left">VaxiJen</th></tr><tr><th align="left">Localization</th><th align="left">Reliability</th><th align="left">SOLpro</th><th align="left">
<bold>PI</bold>
</th><th align="left">Flexibility</th><th align="left"></th></tr></thead><tbody><tr><td align="left" rowspan="2">MrkD</td><td align="left" rowspan="2">AGZ83319.1</td><td align="left">Extracellular</td><td align="left">3.44</td><td align="left" rowspan="2">0.49</td><td align="left" rowspan="2">9.73</td><td align="left" rowspan="2">1.003</td><td align="left" rowspan="2">0.82</td></tr><tr><td align="left">OuterMembrane</td><td align="left">1.49</td></tr><tr><td align="left" rowspan="2">Iron-regulated outer membrane proteins</td><td align="left" rowspan="2">VVJ51509.1</td><td align="left">Extracellular</td><td align="left">4.46</td><td align="left" rowspan="2">0.40</td><td align="left" rowspan="2">4.86</td><td align="left" rowspan="2">1.005</td><td align="left" rowspan="2">0.58</td></tr><tr><td align="left">OuterMembrane</td><td align="left">0.36</td></tr><tr><td align="left" rowspan="2">RmpA</td><td align="left" rowspan="2">AUG89078.1</td><td align="left">Periplasmic</td><td align="left">1.27</td><td align="left" rowspan="2">0.59</td><td align="left" rowspan="2">10.13</td><td align="left" rowspan="2">0.987</td><td align="left" rowspan="2">0.46</td></tr><tr><td align="left">OuterMembrane</td><td align="left">0.91</td></tr><tr><td align="left" rowspan="2">YaiO</td><td align="left" rowspan="2">TNA22439.1</td><td align="left">OuterMembrane</td><td align="left">2.23</td><td align="left" rowspan="2">0.36</td><td align="left" rowspan="2">6.66</td><td align="left" rowspan="2">0.993</td><td align="left" rowspan="2">0.49</td></tr><tr><td align="left">Extracellular</td><td align="left">2.42</td></tr><tr><td align="left" rowspan="2">EMG21</td><td align="left" rowspan="2">TMZ64363.1</td><td align="left">Extracellular</td><td align="left">2.01</td><td align="left" rowspan="2">0.53</td><td align="left" rowspan="2">9.18</td><td align="left" rowspan="2">1.008</td><td align="left" rowspan="2">0.56</td></tr><tr><td align="left">OuterMembrane</td><td align="left">1.43</td></tr><tr><td align="left" rowspan="2">TonB</td><td align="left" rowspan="2">CAA48498.1</td><td align="left">Periplasmic</td><td align="left">3.63</td><td align="left" rowspan="2">0.75</td><td align="left" rowspan="2">10.09</td><td align="left" rowspan="2">1.012</td><td align="left" rowspan="2">0.76</td></tr><tr><td align="left">OuterMembrane</td><td align="left">0.30</td></tr><tr><td align="left" rowspan="2">ompA</td><td align="left" rowspan="2">CDO13836.1</td><td align="left">OuterMembrane</td><td align="left">4.50</td><td align="left" rowspan="2">0.39</td><td align="left" rowspan="2">6.24</td><td align="left" rowspan="2">1.000</td><td align="left" rowspan="2">0.64</td></tr><tr><td align="left">Periplasmic</td><td align="left">0.32</td></tr><tr><td align="left" rowspan="2">ydiY</td><td align="left" rowspan="2">CDO15043.1</td><td align="left">OuterMembrane</td><td align="left">2.23</td><td align="left" rowspan="2">0.58</td><td align="left" rowspan="2">5.02</td><td align="left" rowspan="2">1.004</td><td align="left" rowspan="2">0.69</td></tr><tr><td align="left">Extracellular</td><td align="left">2.42</td></tr></tbody></table></table-wrap>
</p>
</sec>
<sec id="Sec16">
<title>T lymphocyte epitopes chosen</title>
<p id="Par30">Gene database searches led the PSORT II server to predict that three amino acid sequences (MrkD, Iron-regulated outer membrane proteins, and RmpA) were probably located in the plasma membrane. Through CCTOP prediction, we were able to identify 48 extracellular domains.</p>
<p id="Par31">The promiscuous CTL epitopes for 50 alleles were predicted using the IEDB server. The list of HLA alleles used in this study to ascertain the MHC variant sequence is shown in Table S2. Three thousand two hundred twenty-one epitopes with a percentile rank of less than 0.5 for each extracellular domain were found. Based 18 alleles, IEDB predicted 2,107 HTL epitopes with percentile rank &lt; 0.5 and IC50 &lt; 500 nM. Different epitopes have been assigned to several MHC alleles and processed as a single entity in later research.</p>
</sec>
<sec id="Sec17">
<title>Linear B-cell epitopes</title>
<p id="Par32">For each of the 48 receptor complexes, linear B-cell epitopes were found using the ABCpred service. The investigation produced 985 sequences in all.</p>
</sec>
<sec id="Sec18">
<title>The convergence of cellular and humoral stimulatory epitopes</title>
<p id="Par33">The alignment of different sequences facilitates the discovery and choosing of epitopes with the capacity to induce humoral and cellular responses. After the first alignment between B-cell epitopes and CTLs, a list of CTL epitopes overlapped with B-cell patterns was produced. By using class I immunogenicity evaluation, another set of peptides was discovered. The best CTL epitopes were gathered as the final product. Similarly, a list of HTL epitopes overlapping with B-cell genomes was generated and refined by removing those whose IC50 was less than 50 nM. In order to offer 48 matching sequences, the best epitope categories (CTLs and HTLs) were finally aligned.</p>
<p id="Par34">
<table-wrap id="Tab2"><label>Table 2</label><caption><p>Selected CTL and HTL epitopes overlapping with each other and with B-cell epitopes</p></caption><table frame="hsides" rules="groups"><thead><tr><th align="left">Protein Code</th><th align="left">CTL epitope</th><th align="left">HTL epitope</th><th align="left">B-Cell epitope</th><th align="left">interacting MHC-I alleles</th><th align="left">human homology</th></tr></thead><tbody><tr><td align="left" rowspan="13">AGZ83319.1</td><td align="left">---</td><td align="left">---</td><td align="left">ATVNIVYPDVFSSRVYNTTNYSLEGS</td><td align="left">---</td><td align="left">non-homologue</td></tr><tr><td align="left">CTRLS<bold>SPTV</bold></td><td align="left"><bold>SPTV</bold>MLDMVVGRVVV</td><td align="left">S<bold>SPTV</bold>MLDM</td><td align="left"><p>HLA-A*24:02</p><p>HLA-DRB1*03:01</p></td><td align="left">non-homologue</td></tr><tr><td align="left"><bold>TMSAPGG</bold>ASY</td><td align="left">LTRDW<bold>TMSAPGG</bold>ASY</td><td align="left">DW<bold>TMSAPGG</bold></td><td align="left"><p>HLA-B*15:01,</p><p>HLA-DRB1*09:01</p></td><td align="left">non-homologue</td></tr><tr><td align="left">WESGGNPIL</td><td align="left">YDIYVNMQGNVKNNI</td><td align="left">FAAKIVSPGATDLGNKIYSTNV</td><td align="left">HLA-B*40:01, HLA-DRB5*01:01</td><td align="left">non-homologue</td></tr><tr><td align="left">GSVILTRDW</td><td align="left">FMYLYSLNKEMVDER</td><td align="left">ATVNIVYPDVFSSRVYNTTNYSLEGS</td><td align="left">HLA-B*57:01, HLA-DRB5*01:01</td><td align="left">non-homologue</td></tr><tr><td align="left">MSAPGGASY</td><td align="left">KLYFAFLKKNVSRIV</td><td align="left">ATTGSGTL</td><td align="left">HLA-DRB3*02:02, HLA-A*01:01</td><td align="left">non-homologue</td></tr><tr><td align="left">TMSAPGGASY</td><td align="left">AFLKKNVSRIVNHYP</td><td align="left">YDWESGGNPIL</td><td align="left">HLA-B*15:01, HLA-DRB3*02:02</td><td align="left">non-homologue</td></tr><tr><td align="left">MSLRKLLTLF</td><td align="left">GGYKMLRGSLNMISQ</td><td align="left">IRRTDLKGVGTTAGGK</td><td align="left">HLA-DRB1*04:05, HLA-B*57:01</td><td align="left">non-homologue</td></tr><tr><td align="left">GEVESHMIF</td><td align="left">VKNNIEKLYFAFLKK</td><td align="left">SSPTVMLDM</td><td align="left">HLA-B*44:03, HLA-DPA1*01:03/DPB1*04:01</td><td align="left">non-homologue</td></tr><tr><td align="left">RYITIPLHAR</td><td align="left">KFGCKRYIRFMYLYS</td><td align="left">DWTMSAPGG</td><td align="left">HLA-DPA1*02:01/DPB1*05:01, HLA-A*31:01</td><td align="left">non-homologue</td></tr><tr><td align="left">RLNNQETRY</td><td align="left">RYIRFMYLYSLNKEM</td><td align="left">GGLSETGYA</td><td align="left">HLA-A*30:02, HLA-DPA1*01:03/DPB1*04:01</td><td align="left">non-homologue</td></tr><tr><td align="left">YLSANAITV</td><td align="left">FGCKRYIRFMYLYSL</td><td align="left">ATSTTATMGALLNEKAGSGMA</td><td align="left">HLA-A*02:03, HLA-DPA1*02:01/DPB1*05:01</td><td align="left">non-homologue</td></tr><tr><td align="left">ITIPLHARFY</td><td align="left">EKLYFAFLKKNVSRI</td><td align="left">QFNKKYTVGRLNNQETRYI</td><td align="left">HLA-B*57:01, HLA-DPA1*02:01/DPB1*05:01</td><td align="left">non-homologue</td></tr><tr><td align="left" rowspan="11">VVJ51509.1</td><td align="left"><bold>EQQVQGGKE</bold>F</td><td align="left">Q<bold>EIEQQVQGGKE</bold>FKD</td><td align="left">GQEI<bold>EQQVQGGKE</bold></td><td align="left">HLA-A*26:01, HLA-DRB1*12:01</td><td align="left">non-homologue</td></tr><tr><td align="left"><bold>QSDE</bold>DSIIV</td><td align="left">AA<bold>QSDE</bold>DSII VSAN</td><td align="left">AA<bold>QSDE</bold></td><td align="left">HLA-B*15:01, HLA-DQA1*01:01/DQB1*05:01</td><td align="left">non-homologue</td></tr><tr><td align="left">VSGGVRYQW</td><td align="left">TLPALAAQSDEDSII</td><td align="left">YYKSQGDDDYGLWLGKNMSAVTSGGKAYTTDGLNSDRIPGTE</td><td align="left">HLA-B*57:01, HLA-DQA1*04:01/DQB1*04:02</td><td align="left">non-homologue</td></tr><tr><td align="left">NTVDFIGSY</td><td align="left">HPLLLASTLPALAAQ</td><td align="left">YRDESLTFYPFPTLTKGQVSSFSSSQQDTDQ</td><td align="left">HLA-A*26:01, HLA-DPA1*02:01/DPB1*14:01</td><td align="left">non-homologue</td></tr><tr><td align="left">VYYRDESLTF</td><td align="left">NANQMFFDLPQSMAS</td><td align="left">HETFNANQMFFDLPQSMASGGLHNESIYTTGRYPGYS</td><td align="left">HLA-A*23:01, HLA-DQA1*01:01/DQB1*05:01</td><td align="left">non-homologue</td></tr><tr><td align="left">IEIDDNRQL</td><td align="left">LVSQVYYRDESLTFY</td><td align="left">ENRVDDFVGYAQQQDIANGKARSADAIKGGKTDYDNF</td><td align="left">HLA-B*40:01, HLA-DRB3*01:01</td><td align="left">non-homologue</td></tr><tr><td align="left">GSYALPVGK</td><td align="left">DNLRTQLAAYYSTSD</td><td align="left">GVELPDPGKYYGIGKYGAAVNGHLPLISSVNVDDSPLQ</td><td align="left">HLA-A*11:01, HLA-DRB1*12:01</td><td align="left">non-homologue</td></tr><tr><td align="left">QTDGRWQKW</td><td align="left">GDNLRTQLAAYYSTS</td><td align="left">VNRTDMTIDVQSDK</td><td align="left">HLA-B*57:01, HLA-DRB1*12:01</td><td align="left">non-homologue</td></tr><tr><td align="left">HPLLLASTL</td><td align="left">LVSQVYYRDESLTFY</td><td align="left">TDGRWQKWDVTLASP</td><td align="left">HLA-DRB1*04:01, HLA-B*07:02</td><td align="left">non-homologue</td></tr><tr><td align="left">ETFNANQMF</td><td align="left">FFGQNLVSQVYYRDE</td><td align="left">PDPWSLRVQSQQVFDLSDAAGNKLEG</td><td align="left">HLA-A*26:01, HLA-DRB3*02:02</td><td align="left">non-homologue</td></tr><tr><td align="left">LQGIKVNSY</td><td align="left">GYSISNVAPFLQSSY</td><td align="left">YVTIWGQRAPLLYSPTYGSSSLYEYKG</td><td align="left"><p>HLA-DRB1*09:01,</p><p>HLA-B*15:01</p></td><td align="left">non-homologue</td></tr><tr><td align="left" rowspan="8">AUG89078.1</td><td align="left">
<bold>LTDDYFFYY</bold>
</td><td align="left">NIV<bold>LTDDYFFYY</bold>GLK</td><td align="left">---</td><td align="left">HLA-A*01:01, HLA-DRB3*01:01</td><td align="left">non-homologue</td></tr><tr><td align="left">
<bold>KLYFAFLKK</bold>
</td><td align="left">E<bold>KLYFAFLKK</bold>NVSRI</td><td align="left">---</td><td align="left">HLA-A*03:01, HLA-DRB1*11:01</td><td align="left">non-homologue</td></tr><tr><td align="left">
<bold>ITYEGVVNK</bold>
</td><td align="left">FH<bold>ITYEGVVNK</bold>SIAI</td><td align="left"><bold>VVNK</bold>SIAIK</td><td align="left"><p>HLA-A*11:01, HLA-A*03:01,</p><p>HLA-DRB5*01:01</p></td><td align="left">non-homologue</td></tr><tr><td align="left">GSLNMISQW</td><td align="left">EKLYFAFLKKNVSRI</td><td align="left">GGINEIKSQLKIEEKT</td><td align="left">HLA-B*57:01, HLA-DRB1*11:01</td><td align="left">non-homologue</td></tr><tr><td align="left">KSIAIKHKR</td><td align="left">LYFAFLKKNVSRIVN</td><td align="left">KEMVDERWLM</td><td align="left">HLA-A*31:01, HLA-DRB1*11:01</td><td align="left">non-homologue</td></tr><tr><td align="left">DVSGGGRFY</td><td align="left">GGYKMLRGSLNMISQ</td><td align="left">SGGGRFYPKGCDYDIYVNMQGNVKNN</td><td align="left">HLA-A*26:01, HLA-DRB1*01:01</td><td align="left">non-homologue</td></tr><tr><td align="left">KTLSCYQSK</td><td align="left">FFYYGLKQLTGLPLF</td><td align="left">SGKWGGYKMLRGSLNMIS</td><td align="left">HLA-DPA1*02:01/DPB1*01:01</td><td align="left">non-homologue</td></tr><tr><td align="left">LNMISQWMW</td><td align="left">LQCLLKNGGINEIKS</td><td align="left">SRIVNHYPRLTKK</td><td align="left">HLA-B*58:01, HLA-DRB1*13:02</td><td align="left">non-homologue</td></tr></tbody></table></table-wrap>
</p>
</sec>
<sec id="Sec19">
<title>Human homology and epitope conservation</title>
<p id="Par35">Every chosen MHC class epitope showed 100% maximal identity for conservation. Additionally, selected CTL, HTL, and LBL epitopes have been demonstrated to be non-homologous to the human amino acid database, corroborating the epitopes’ classification as antigens or exogenous materials inside the human body. Table <xref ref-type="table" rid="Tab2">2</xref> presents studies of human homology and conservation for the selected epitopes.</p>
</sec>
<sec id="Sec20">
<title>Allele and CTL epitope molecular docking analyses</title>
<p id="Par36">It would help to search for a protein structure whose chemical makeup is comparable to that of the crystalline ligand to get reliable binding data. The alleles with suitable interactions with our receptor were chosen for the docking study. The proper allelic interactions were determined on the IEDB website. The binding impact of the chosen CTL epitopes with their respective HLA alleles was examined using molecular docking modelling. We selected the HLA-A*24:02 and HLA-DRB1*03:01 alleles for the CTRLSSPTV, SPTVMLDMVVGRVVV, and SSPTVMLDM epitopes of the MrkD protein. HLA-A*26:01 and HLA-DRB1*12:01 were selected for the EQQVQGGKEF, QEIEQQVQGGKEFKD, and GQEIEQQVQGGKE epitopes of iron-regulated outer membrane proteins. A*11:01, A*03:01, and HLA-DRB5*01:01 were selected for the RmpA protein’s ITYEGVVNK, FHITYEGVVNKSIAI, and VVNKSIAIK epitopes. Furthermore, as Fig. <xref ref-type="fig" rid="Fig1">1</xref> illustrates, distinct alleles were selected for the epitopes of RmpA, MrkD, and Iron-regulated outer membrane proteins.</p>
<p id="Par37">Crystallized ligand epitopes were used as the positive control to validate the docking experiments. Molecular docking results connected the HLA alleles, specific CTL epitopes, and positive controls. All positive controls had higher binding affinities than other specific CTL epitopes, except the HLA-A*24:02 control, where TMSAPGGASY (-10 kcal/mol) and LTRDWTMSAPGGASY (-9.2 kcal/mol) epitopes produced higher docking amounts than the positive control (-9.1 kcal/mol) of the HLA-A*24:02 allele. They possessed fewer hydrogen bond linkages than the positive control of HLA-A*24:02, even though their docking values were greater. The interactions induced among the specific alleles and the favorable controls were similar to those between the CTL epitopes and the particular ligands. With the HLA-A*26:01 alleles, EQQVQGGKEF produced the most significant number of hydrogen bonds—eight—while the control HLA-A*01:01 allele revealed the most significant number of hydrogen bonds—12. The positive control HLA-B*14:02 (-10.0 kcal/mol) exhibited higher docking scores than the epitopes LQGIKVNSY (-8.2 kcal/mol) and LNMISQWMW (-7.6 kcal/mol). However, these epitopes formed more hydrogen bonds than the control group. Furthermore, hydrophobic linkages allowed the CTL epitopes and positive controls to engage with the receptors.</p>
</sec>
<sec id="Sec21">
<title>Evaluation of population coverage</title>
<p id="Par38">In the population coverage investigation, the global coverage of the MrkD antigen MHCI and class combination epitopes was 54.01% and 59.28%, respectively. According to the community prevalence study, the combined epitopes of the MHC class and the iron-regulated outer membrane proteins MHCI showed 53.74% and 48.66% worldwide coverage. Table S3, <ext-link ext-link-type="uri" xlink:href="https://s24.picofile.com/file/8454030850/S3_TABLE.docx.html">https://s24.picofile.com/file/8454030850/S3_TABLE.docx.html</ext-link>, shows the results of the population coverage study for the RmpA protein MHCI and class combination epitopes, which showed 44.37% and 50.79% worldwide coverage, respectively. We focused on the combined population coverage of both MHC epitopes in vaccine proteins. 79% of the world’s population was covered. The regions with the highest population coverage for combined MHC Class-I and class-II epitopes were Oceania (79.22%), North America (82.12%), West Africa (75.19%), Northeast Asia (81.21%), North Africa (86.05%), South Africa (74.56%), South Asia (64.38%), East Africa (52.98%), Southwest Asia (57.20%), West Indies (74.56%), Central Africa (81.09%), Southeast Asia (53.71%), East Asia (76.27%), South America (61.22%), and Central America (51.22%) (Fig. <xref ref-type="fig" rid="Fig3">3</xref>).</p>
<p id="Par39">
<fig id="Fig3"><label>Fig. 3</label><caption><p>Population Coverage Calculation Result for MrkD, Iron-regulated outer membrane proteins and RampA epitopes. the results of the population coverage study for the RmpA protein MHCI and class combination epitopes, which showed 44.37% and 50.79% worldwide coverage, respectively</p></caption><graphic id="d33e1345" xlink:href="12865_2024_617_Fig3_HTML"></graphic></fig>
</p>
</sec>
<sec id="Sec22">
<title>MHC cluster evaluation results</title>
<p id="Par40">The MHCcluster v2.0 database categorized the MHCI and MHCII variants based on their interactions with the designated structural proteins’ epitopes. Twenty-one alleles from each MHC class are included in the study: HLA-B15:01, HLA-A02:05, HLA-A01:01, HLA-A02:07, HLA-A30:01, HLA-B27:02, HLA-A66:01, HLA-A68:02, HLA-B07:02, HLA-B18:01, HLA-B27:05, HLA-A31:01, HLA-B45:01, HLA-B51:01, HLA-B55:01, HLA-B57:01, HLA-C03:03, HLA-C04:01, HLA-C08:01, and HLA-C12:02 in each MHC class. Figure <xref ref-type="fig" rid="Fig4">4</xref>A and C illustrate the clustering pattern for the MHCI and MHCII alleles. An enhanced tree diagram of the MHCI and MHCII clustering results is shown in Fig. <xref ref-type="fig" rid="Fig4">4</xref>B, D and E, and <xref ref-type="fig" rid="Fig4">4</xref>F. The heatmap’s red areas revealed stronger linkages between the concentrated HLA alleles, while its yellow areas revealed weaker correlations.</p>
<p id="Par41">
<fig id="Fig4"><label>Fig. 4</label><caption><p>The findings of the MHC clustering algorithms. Heatmap of MHC class-I cluster assessment of (<bold>A</bold>) MrkD, (<bold>C</bold>) Iron-regulated outer membrane proteins, and (<bold>E</bold>) RmpA protein. Advanced tree map of MHC Class-I cluster evaluation of (<bold>B</bold>) MrkD, (<bold>D</bold>) Iron-regulated outer membrane proteins, and (<bold>F</bold>) RmpA protein. The read portions of the heatmap showed higher relationships between the clustered HLA alleles, while the heatmap’s yellow sections showed lesser connections between clustering HLA alleles</p></caption><graphic id="d33e1388" xlink:href="12865_2024_617_Fig4_HTML"></graphic></fig>
</p>
</sec>
<sec id="Sec23">
<title>The Vaccine Construct is being developed</title>
<p id="Par42">The most potent LBL, CTL, and HTL epitopes were combined with HEYGAEALERAG and GGGS linkers to create the vaccine design. The adjuvant CPG ODN C274 (5’-TCGTCGAACGTTCGAGATGAT-3’) was attached to the front of the vaccination using the EAAAK linker. A CpG ODN C274-mediated rat TLR-9 stimulation in NF-kB LeeporterTM—RAW 264.7 cells promote overall IgG antibodies, Th1 cell protection, and the TLR-9 pathway. From the structural protein sequences of <italic>K. pneumoniae</italic>, seven CTL, seven HTL, and seven LBL epitopes were selected for the present investigation, respectively. After meticulous combining and randomizing, we designed a vaccine. Table <xref ref-type="table" rid="Tab3">3</xref> displays the completed vaccination build.</p>
<p id="Par43">
<table-wrap id="Tab3"><label>Table 3</label><caption><p>Sequence of vaccine structures designed in this study</p></caption><table frame="hsides" rules="groups"><thead><tr><th align="left">Epitope composition</th><th align="left">group</th><th align="left">Vaccine sequence</th><th align="left">AlgPred</th><th align="left">SOLpro (&lt; 0.4)</th><th align="left">PI</th><th align="left">GRAVY</th><th align="left">VaxiJen</th><th align="left">Homology</th></tr></thead><tbody><tr><td align="left" rowspan="2">Multi epitope for KELEBSIELLA</td><td align="left">Adjuvant + 7CTL + 7HTL + 7LBL</td><td align="left">MSSNVRDDSSNVRDDSSNVRDDSSNVRDDSSNVRDDEAAAKCTRLSSPTVHEYGAEALERAGTMSAPGGASYGGGSEQQVQGGKEFGGGSQSDEDSIIVGGGSLTDDYFFYYGGGSKLYFAFLKKGGGSITYEGVVNKGGGSSPTVMLDMVVGRVVVGGGSLTRDWTMSAPGGASYGGGSQEIEQQVQGGKEFKDGGGSAAQSDEDSIIVSANGGGSNIVLTDDYFFYYGLKGGGSEKLYFAFLKKNVSRIGGGSFHITYEGVVNKSIAIGGGSATVNIVYPDVFSSRVYNTTNYSLEGSGGGSATVNIVYPDVFSSRVYNTTNYSLEGSGGGSYYKSQGDDDYGLWLGKNMSAVTSGGKAYTTDGLNSDRIPGTEGGGSHETFNANQMFFDLPQSMASGGLHNESIYTTGRYPGYSGGGSENRVDDFVGYAQQQDIANGKARSADAIKGGKTDYDNFGGGSVVNKSIAIKHEYGAEALERAGSGKWGGYKMLRGSLNMISEAAAK</td><td align="left">NON ALLERGEN</td><td align="left">0.516</td><td align="left">4.99</td><td align="left">-0.33</td><td align="left">1.13</td><td align="left">non-homologue</td></tr><tr><td align="left">Adjuvant + 7CTL + 7HTL + 7LBL + Adjuvant</td><td align="left">MCRTFEMEAAAKCTRLSSPTVHEYGAEALERAGSPTVMLDMVVGRVVVGGGSATVNIVYPDVFSSRVYNTTNYSLEGSGGGSTMSAPGGASYGGGSLTRDWTMSAPGGASYGGGSATVNIVYPDVFSSRVYNTTNYSLEGSGGGSEQQVQGGKEFGGGSQEIEQQVQGGKEFKDGGGSYYKSQGDDDYGLWLGKNMSAVTSGGKAYTTDGLNSDRIPGTEGGGSQSDEDSIIVGGGSQEIEQQVQGGKEFKDGGGSHETFNANQMFFDLPQSMASGGLHNESIYTTGRYPGYSGGGSLTDDYFFYYGGGSNIVLTDDYFFYYGLKGGGSENRVDDFVGYAQQQDIANGKARSADAIKGGKTDYDNFGGGSKLYFAFLKKGGGSEKLYFAFLKKNVSRIGGGSVVNKSIAIKGGGSITYEGVVNKGGGSFHITYEGVVNKSIAIHEYGAEALERAGSGKWGGYKMLRGSLNMISEAAAKCRTFEM.</td><td align="left">NON ALLERGEN</td><td align="left">0.506</td><td align="left">5.11</td><td align="left">-0.38</td><td align="left">1.13</td><td align="left">non-homologue</td></tr></tbody></table></table-wrap>
</p>
</sec>
<sec id="Sec24">
<title>Antigenicity, allergenicity, and physicochemical profile of vaccines</title>
<p id="Par44">The Expasy website assessed the physicochemical characteristics of the produced component. Vaccine-1’s molecular formula was found to be C2244H3428N622O764S16. The vaccine-2 form has 39 positively charged residues (Arg + Lys) and 49 negatively charged residues (Asp + Glu), with a molecular weight of 51856.57 Da, indicating a mean mass. Using the Expasy website, the chemical formula for vaccine-2 was C2262H3453N627O766S16. The vaccination-2 construct has a molecular weight of 52200.00 Da, 50 negatively charged residues (Asp + Glu), and 39 positively charged residues (Arg + Lys). For both vaccines, the predicted half-lives were 7.2 h (for mammalian reticulocytes In vitro), &gt; 20 h (for yeast In vivo), and &gt; 10 h (for <italic>Escherichia coli</italic> In vivo).</p>
<p id="Par45">For vaccines 1 and 2, the expected PI characteristics of 4.99 and 5.11 indicated that the vaccination was alkaline, respectively. Vaccine-1 had an aliphatic index value of 57.04 and an instability value of 40.02. Vaccine-2’s aliphatic index value was 55.58, and the instability score was 41.05. The structure was hydrophilic (GRAVY) based on the grand mean of hydropathicity values (vaccine-1: -0.33, vaccine-2: -0.38). The vaccines’ antigenicity and allergenicity were developed. Using the Vaxijen2.0 server, the antigenicity values for vaccines-1 and vaccine-2 were found to be 1.13 and 1.1364, respectively. Interestingly, all servers demonstrated that the immunizations were non-allergic and antigenic. Additionally, the Protein-Sol server was soluble in the vaccine design, which gave vaccine-1 and vaccine-2 values of 0.516 and 0.506, respectively (Table <xref ref-type="table" rid="Tab3">3</xref>). Moreover, the corresponding prediction results for the vaccine constructions did not include TM helices or signal peptides.</p>
</sec>
<sec id="Sec25">
<title>Blast homology analysis</title>
<p id="Par46">Table <xref ref-type="table" rid="Tab3">3</xref> shows that there was no similarity (0%) between the vaccine construct’s query coverage and the proteins of Homo sapiens, based on the sequence homology between the generated vaccination protein sequence and the human proteome sequence. The anticipated vaccine protein will not cause autoimmune responses in the host, according to the results of the BLAST homology study.</p>
</sec>
<sec id="Sec26">
<title>Prediction of secondary and 3D structure of vaccines</title>
<p id="Par47">We made use of internet servers to assess the secondary structural aspects. Based on PSIPRED service projections, the final vaccine-1 construct would have 30.52% α-helix, 16.13% β-strand, and 53.13% random coils; the vaccine-2 construct would have 29.85% α-helix, 15.71% β-strand, and 51.47% random coils among its constituent parts. Ramachandran’s analysis shows that vaccine-1 has no bad bonds (out of 885). However, table S4 (<ext-link ext-link-type="uri" xlink:href="https://s25.picofile.com/file/8454143276/S4_TABLE.docx.html">https://s25.picofile.com/file/8454143276/S4_TABLE.docx.html</ext-link>) shows that there are 14 bad angles (A196 ALA, A149 ASP), (A207 ASP-A208 ASP), A155 PRO, (A151 THR-A152 MET), (A154 ALA-A155 PRO), (A265 TYR-A266 PRO, A208 ASP, A220 SER, A241 HIS, A240 PHE, A179 ASP). The results of Ramachandran’s study indicate that vaccine-2 contains two bad bonds (out of 696 bonds) and fourteen bad angles (B328 THR, B326 VAL, B302 TYR, B316 PHE, B356 GLN, B377 ASP, B387 PHE, B329 ASP, B319 TYR, B357 ASP, B313 ASP, B373 LYS, B338 LYS, (B338 LYS-B339 GLY), B328 THR, B326 VAL, and B357 ASP) (Fig. <xref ref-type="fig" rid="Fig5">5</xref>).</p>
<p id="Par51">
<fig id="Fig5"><label>Fig. 5</label><caption><p>Ramachandran’s analysis. (<bold>A</bold>) vaccine-1 construct had no bad bond, 14 Bad Angles (<bold>B</bold>) vaccine-2 construct had 2 bad bonds in position of B357 ASP, B326 VAL and 14 Bad Angles</p></caption><graphic id="d33e1521" xlink:href="12865_2024_617_Fig5_HTML"></graphic></fig>
</p>
<p id="Par48">The quality of Vaccination 1’s third structure was examined using validation techniques like ERRAT and Verify 3D. Figure <xref ref-type="fig" rid="Fig6">6</xref>A of the ERRAT findings demonstrated an average quality factor of 93.8596. Furthermore, in the 3D/1D profile of Verify 3D, at least 80% of the amino acids scored &gt; = 0.1, and 91.72% of the residues had an average 3D-1D score &gt; = 0.1, indicating that the third structure was approved (Fig. <xref ref-type="fig" rid="Fig6">6</xref>B). However, All-residue chi1-chi2 plots and All-residue Ramachandran plots both passed (Fig. <xref ref-type="fig" rid="Fig7">7</xref>). Figure <xref ref-type="fig" rid="Fig7">7</xref> displayed the chi1-chi2 plots for each residual. Within parenthesis, the total amount of residues is shown. Labels were applied to those with poor combinations (scoring &lt; -3.00). Positive conformations found via examination of 163 compounds at 2.0 A resolution or above are shown by shading (Fig. <xref ref-type="fig" rid="Fig7">7</xref>).</p>
<p id="Par52">
<fig id="Fig6"><label>Fig. 6</label><caption><p>(<bold>A</bold>) The ERRAT results showed an overall quality factor of 93.8596. (<bold>B</bold>) at least 80% of the amino acids have scored &gt; = 0.1 in the 3D/1D profile of Verify 3D</p></caption><graphic id="d33e1555" xlink:href="12865_2024_617_Fig6_HTML"></graphic></fig>
</p>
<p id="Par53">
<fig id="Fig7"><label>Fig. 7</label><caption><p>All-residue chi1-chi2 plots. Numbers of residues are shown in brackets. Those in unfavorable conformations (score &lt; -3.00) are labelled. Shading shows favorable conformations as obtained from an analysis of 163 structures at resolution 2.0 A or better</p></caption><graphic id="d33e1565" xlink:href="12865_2024_617_Fig7_HTML"></graphic></fig>
</p>
<p id="Par49">Molecular dynamics (MD) simulation trajectories can yield plots useful for assessing a system’s structural stability. By the end of our equilibrium period, the density, temperature, and total energy graphs had reached a state of convergence, suggesting the stability of the vaccine structure. As our equilibration time concludes, the density, temperature, and total energy graphs have all converged. Even though it seems to be leveling off, the RMSD is still tolerable and has not entirely converged (Fig. <xref ref-type="fig" rid="Fig8">8</xref>A). The JCAT findings demonstrated that, with adaptation, the vaccination sequence was appropriate for mouse muscle. Furthermore, as shown in Fig. <xref ref-type="fig" rid="Fig8">8</xref>B, the enhanced sequence’s GC content was similar to 67.06192358366272, and its CAI value was equal to 0.70980977263218006. Once it was chosen, the vaccine gene sequence of choice was introduced into the pcDNA3.1(+) vaccine construct. This project’s route map is displayed in Fig. <xref ref-type="fig" rid="Fig8">8</xref>C.</p>
<p id="Par54">
<fig id="Fig8"><label>Fig. 8</label><caption><p>(<bold>A</bold>) Graphs generated from molecular dynamics (MD) simulation trajectories can be used to analyses a system’s structural stability. The density, temperature and total energy plots have all converged by the end of our equilibration period. The RMSD, while seeming to begin to level, does not have converged entirely but is acceptable. (<bold>B</bold>) JCAT results showed that the vaccine sequence was suitable for mouse muscle after adaptation with GC-Content of 6706%. (<bold>C</bold>) Map of pcDNA3.1(+) showing the multi-epitope vaccine locus</p></caption><graphic id="d33e1596" xlink:href="12865_2024_617_Fig8_HTML"></graphic></fig>
</p>
<p id="Par50">Figure <xref ref-type="fig" rid="Fig9">9</xref> depicts the molecular docking analysis of the TLR-9 receptor and vaccination epitope. The proposed vaccine was subjected to molecular docking to TLR-9 (PDB ID-3WPF) utilizing the ClusPro v 2.0 and Patchdock web servers. Consequently, ClusPro v 2.0 produced 29 structures for each docking. In both instances, the model with the highest binding affinity and lowest intermolecular energy was chosen. The docking process with TLR9 resulted in the lowest energy score of -1163, as shown in Fig. <xref ref-type="fig" rid="Fig9">9</xref>. The toxicity of the final epitopes in the vaccine structure was predicted and the results showed the non-toxicity of the vaccine structure (Table S5).</p>
<p id="Par55">
<fig id="Fig9"><label>Fig. 9</label><caption><p>Analysis of molecular docking between the TLR-9 receptor and vaccine epitope. The results showed a strong binding between the TLR-9 receptor and the vaccine construct</p></caption><graphic id="d33e1614" xlink:href="12865_2024_617_Fig9_HTML"></graphic></fig>
</p>
</sec>
</sec>
<sec id="Sec27" sec-type="discussion">
<title>Discussion</title>
<p id="Par56">The use of whole organisms is a fundamental aspect of conventional vaccine manufacturing techniques, which have the potential to induce allergic responses and inadvertent antigen exposure [<xref ref-type="bibr" rid="CR17">17</xref>]. Polypeptide vaccinations provide a potential solution to the issue of allergic reactions by eliminating the possibility of such reactions. These vaccines contain short segments of immunogenic peptides and induce robust and specific immune responses [<xref ref-type="bibr" rid="CR18">18</xref>]. The prevalence of immunizations against infectious illnesses is on the rise and is very effective. In order to ensure the effectiveness of vaccines, the preclinical and clinical stages of vaccine development need the use of intricate, time-consuming, and costly in vivo and in vitro methodologies [<xref ref-type="bibr" rid="CR19">19</xref>, <xref ref-type="bibr" rid="CR20">20</xref>].</p>
<p id="Par57">The efficiency of in silico vaccine development is increasing due to advancements in bioinformatics and immunoinformatic [<xref ref-type="bibr" rid="CR21">21</xref>]. This phenomenon results in a decline in In vitro testing procedures. The advancement of a multiepitope vaccine targeting K. pneumoniae was significantly facilitated by an in vitro investigation. The primary benefit of this technology is its ability to detect many candidate vaccinations without spreading bacterial illnesses across a wet laboratory [<xref ref-type="bibr" rid="CR22">22</xref>]. Several instances of how vaccinosis has been used to construct a multi-epitope-based vaccination method include Acinetobacter baumannii [<xref ref-type="bibr" rid="CR16">16</xref>], MERS-CoV, HIV, and SARS-CoV-2 [<xref ref-type="bibr" rid="CR22">22</xref>–<xref ref-type="bibr" rid="CR25">25</xref>].</p>
<p id="Par58">Klebsiella is responsible for many diseases, including potentially fatal infections affecting the liver, circulation, and lungs. The absence of an FDA-approved vaccination that effectively prevents Klebsiella infections necessitates the urgent proposal of new vaccine candidates for immune protection studies [<xref ref-type="bibr" rid="CR26">26</xref>]. The standard methodology for vaccine design is characterized by its high cost and significant time investment. Computational techniques can accelerate vaccine development by identifying potent vaccine candidates versus Klebsiella [<xref ref-type="bibr" rid="CR27">27</xref>]. K. pneumoniae, being a diverse organism, continuously uncovers novel serotypes. Further research is required to identify a potential vaccine candidate to target a diverse array of K. pneumoniae effectively isolates [<xref ref-type="bibr" rid="CR28">28</xref>].</p>
<p id="Par59">The current work is unique in that it utilizes genomes to uncover a widely recognized vaccination candidate for many strains of K. pneumoniae rather than concentrating on a single organism. This kind of study is often called “pan-genome reverse vaccinology.” When developing a vaccine candidate, it is necessary to consider many microbial strains [<xref ref-type="bibr" rid="CR29">29</xref>]. Researchers used this technique to clone and evaluate eight strains of group B streptococcus to identify vaccine candidates with high efficacy [<xref ref-type="bibr" rid="CR30">30</xref>]. A methodology that integrates wet and dry laboratories was used to select possible vaccine candidates against diseases by analyzing the genome of a single strain. This approach facilitated the development of novel vaccine candidates targeting the bacteria [<xref ref-type="bibr" rid="CR31">31</xref>].</p>
<p id="Par60">This methodology has undergone testing against bacterial pathogens and a diverse range of other hazardous species. The efficiency of this strategy against the protozoan Leishmania was established by researchers using proteome screening. This was achieved using relevant properties such as subcellular localization, non-homology to the protein database, and ligand binding with MHC [<xref ref-type="bibr" rid="CR32">32</xref>]. Another approach included the selection of a well-known antigenic spike peptide from the SARS coronavirus and subsequent analysis of its sequence to find the regions with the highest antigenicity. The decrease in energy improved these areas, resulting in increased antigenic characteristics [<xref ref-type="bibr" rid="CR33">33</xref>]. Eleven thermodynamically stable epitopes, which exhibit similarity to natural proteins, were identified using molecular modeling investigations utilizing glycoconjugate addition on 14 epitopes derived from <italic>Schistosoma mansoni</italic>. The potential use of these epitopes as vaccine targets against schistosomiasis has been explored in wet laboratory trials [<xref ref-type="bibr" rid="CR34">34</xref>].</p>
<p id="Par61">A web-based approach was used to identify candidates from a pool of 10 surface-exposed adhesion molecules of Staphylococcus aureus known to cause endocarditis. This technique included the search for an epitope that exhibited conservation across different bacterial strains and showed the ability to induce humoral and cell-mediated protection [<xref ref-type="bibr" rid="CR35">35</xref>]. With the continuous identification of whole genome sequences of disease-causing microorganisms and the genomes of specific surface antigens, it is now possible to rapidly test a specific antigen or group of proteins using a simple technical application. Reverse vaccination is a cost-effective strategy due to the extensive accessibility of genetic data and most screening tools. The reversal of vaccination has the potential to effectively cure a wide range of bacterial, fungal, viral, and parasitic diseases. The BLAST technology used in this approach applies to a wide range of organisms, including fungi and viruses. The analysis of input sequences in the IEDB B cell epitope identification software does not consider the kind of pathogen. Cello protein, a protein localization instrument, may be used to forecast the subcellular positioning of proteins [<xref ref-type="bibr" rid="CR36">36</xref>].</p>
<p id="Par62">In addition to its many benefits, reverse vaccination has significant drawbacks. The availability of complete genetic information about the pathogen is crucial for verifying vaccination candidates, while in vivo experiments should validate particular antigens [<xref ref-type="bibr" rid="CR37">37</xref>, <xref ref-type="bibr" rid="CR38">38</xref>]. The study’s primary aim was to ascertain the proteins of K. pneumoniae that exhibit the highest degree of conservation and potential antigenicity via web-based bioinformatics methodologies. The pathogenicity and antigenic potential of pneumococcal surface antigens have been extensively investigated in animal experiments. Proteins such as pspA and pspC have been shown to promote resistance in mouse models [<xref ref-type="bibr" rid="CR39">39</xref>].</p>
<p id="Par63">Recombinant pneumococcal glycolytic proteins, including GAPDH (Glyceraldehyde 3-phosphate dehydrogenase) and FBA [<xref ref-type="bibr" rid="CR40">40</xref>], are produced inside Escherichia coli and elicit the production of antibodies that exhibit specificity towards various serotypes. Research [<xref ref-type="bibr" rid="CR41">41</xref>] shows an observed interaction between pneumococcal serotypes and vaccinations targeting choline-binding enzymes (CBPs). Following the administration of spuA and phpA vaccines [<xref ref-type="bibr" rid="CR42">42</xref>], the body generates antibodies against pneumococci. The study utilized EAAAK, GGGS, and HEYGAEALERAG linkers to establish connections between various vaccine components. These linkers are crucial in determining both B and T cells’ 3D structure and subsequent exposure [<xref ref-type="bibr" rid="CR43">43</xref>, <xref ref-type="bibr" rid="CR44">44</xref>]. The “EAAAK” linker attached the Adjuvant to vaccination sequences. This linker is an inflexible linker, which offers some advantages compared to flexible linkers. The “EAAAK” linker in a bi-functional fusion protein facilitates the efficient segregation of functional domains [<xref ref-type="bibr" rid="CR45">45</xref>]. The HTL epitopes were connected using the “GGGS” linker. This linker can trigger HTL (helper T lymphocyte) responses, essential for creating multi-epitope vaccines. This linker inhibits the production of the majority of junctional epitopes [<xref ref-type="bibr" rid="CR46">46</xref>]. The “HEYGAEALERAG” linker was utilized to join CTL epitopes. This linker serves as the target for both the lysosomal and proteasomal degradation processes, enhancing the presentation of epitopes.</p>
</sec>
<sec id="Sec28" sec-type="conclusion">
<title>Conclusions</title>
<p id="Par64">This study used a computational vaccine design approach to all <italic>Klebsiella</italic> genus species reference genomes. Three exceptionally conserved proteins and putative vaccination candidates that fit several vaccine candidacy traits were found. MrkD, RmpA, and iron-regulated outer membrane proteins are these proteins. Several highly rated epitopes from the epitope mapping phase of these proteins were used to create a multi-epitope peptide vaccination. The created vaccine ensemble also showed a prolonged binding configuration with the evaluated immune receptor and significant interaction energy. The time and expense of developing vaccines might be significantly reduced using computer-aided design. The process can also aid in identifying the most suitable vaccine candidates for examination. The study’s results are promising, but further research is needed to confirm the vaccine architecture’s biological efficacy. Despite its early stages, the present research is more economical and time-efficient than a typical method for screening potential vaccines; nonetheless, a biological test is still required to confirm the screening results.</p>
</sec>
<sec sec-type="supplementary-material">
<sec id="Sec29">
<title>Electronic supplementary material</title>
<p>Below is the link to the electronic supplementary material.</p>
<p>
<supplementary-material content-type="local-data" id="MOESM1"><media xlink:href="12865_2024_617_MOESM1_ESM.docx"><caption><p>Supplementary Material 1</p></caption></media></supplementary-material>
</p>
</sec>
</sec>
</body>